\documentclass[10pt]{article}
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL MS_orig.tex      Wed Nov 12 10:08:35 2014
%DIF ADD MS_revised.tex   Wed Nov 12 10:08:20 2014
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{cite}
%DIF 5c5-6
%DIF < \usepackage{hyperref}
%DIF -------
%\usepackage{hyperref} %DIF > 
\usepackage[hidelinks]{hyperref} %DIF > 
%DIF -------
\usepackage{microtype}
%\usepackage{lineno}
%DIF 8a9
%\linenumbers %DIF > 
%DIF -------
\DisableLigatures[f]{encoding = *, family = * }

% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

%DIF 21-22d23
%DIF < % Use the PLoS provided BiBTeX style
%DIF < \bibliographystyle{plos2009}
%DIF -------

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}

%% Include all macros below. Please limit the use of macros.

%% END MACROS SECTION
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFaddtex}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdeltex}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF HYPERREF PREAMBLE %DIF PREAMBLE
\providecommand{\DIFadd}[1]{\texorpdfstring{\DIFaddtex{#1}}{#1}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{\texorpdfstring{\DIFdeltex{#1}}{}} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}

\begin{flushleft}
{\Large
    \title{Diversity dynamics in Nymphalidae butterflies: Effect of phylogenetic uncertainty on diversification rate shift estimates}
}
\maketitle
Carlos Pe\~na$^{1,\ast}$,
Marianne Espeland$^{2}$
\\
\bf{1} Carlos Pe\~na Laboratory of Genetics, Department of Biology, University of Turku, Turku, Finland. Email: \href{mailto:mycalesis@gmail.com}{\nolinkurl{mycalesis@gmail.com}}
\\
\bf{2} Marianne Espeland Museum of Comparative Zoology and Department
of Organismic and Evolutionary Biology, Harvard University, Cambridge,
USA. Email:
\href{mailto:marianne.espeland@gmail.com}{\nolinkurl{marianne.espeland@gmail.com}}.
Current address: McGuire Center for Lepidoptera and Biodiversity,
Florida Museum of Natural History, University of Florida, Gainesville,
FL, US.
\\
$\ast$ \textbf{Corresponding author:}
\href{mailto:mycalesis@gmail.com}{\nolinkurl{mycalesis@gmail.com}}
Telephone: +358 417063065 Fax: +358 2 333 6680

\DIFdelbegin \textbf{\DIFdel{Comments:}} %DIFAUXCMD
\DIFdel{5814 words, 7 figures, 2 tables and 25 supplementary
material files.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\textbf{\DIFdel{Additional Supplementary materials:}}
%DIFAUXCMD
%DIFDELCMD < \url{http://dx.doi.org/10.6084/m9.figshare.639208}
%DIFDELCMD < %%%
\DIFdelend \end{flushleft}




\subsubsection*{Running title: Diversity dynamics in Nymphalidae
butterflies}


\section*{Abstract}

The species rich butterfly family Nymphalidae has been used to study
evolutionary interactions between plants and insects. Theories of
insect-hostplant dynamics predict accelerated diversification due to key
innovations. \DIFdelbegin \DIFdel{We investigated whether phylogenetic uncertainty affects a
commonly used method (MEDUSA, }\DIFdelend \DIFaddbegin \DIFadd{In evolutionary biology, analysis of maximum credibility
trees in the software MEDUSA (}\DIFaddend modelling evolutionary diversity using
stepwise AIC) \DIFdelbegin \DIFdel{for estimating }\DIFdelend \DIFaddbegin \DIFadd{is a popular method for estimation of }\DIFaddend shifts in
diversification rates\DIFdelbegin \DIFdel{in
lineages, }\DIFdelend \DIFaddbegin \DIFadd{. We investigated whether phylogenetic uncertainty
can produce different results }\DIFaddend by extending the method across a random
sample of trees from the posterior distribution of a Bayesian run. \DIFdelbegin \DIFdel{We }\DIFdelend \DIFaddbegin \DIFadd{Using
the MultiMEDUSA approach, we }\DIFaddend found that phylogenetic uncertainty greatly
affects diversification rate estimates. Different trees produced
diversification rates ranging from high values to almost zero for the
same clade, and both significant rate increase and decrease in some
clades. Only four out of 18 significant shifts found on the maximum
clade credibility tree were consistent across most of the sampled trees.
Among these, we found accelerated diversification for Ithomiini
butterflies. We used the binary speciation and extinction model (BiSSE)
and found that a hostplant shift to Solanaceae is correlated with
increased net diversification rates in Ithomiini, congruent with the
diffuse cospeciation hypothesis. Our results show that taking
phylogenetic uncertainty into account when estimating net
diversification rate shifts is of great importance, \DIFdelbegin \DIFdel{and relying on }\DIFdelend \DIFaddbegin \DIFadd{as very different
results can be obtained when using }\DIFaddend the maximum clade credibility tree
\DIFdelbegin \DIFdel{alone can potentially give erroneous
results}\DIFdelend \DIFaddbegin \DIFadd{and other trees from the posterior distribution}\DIFaddend .

\textbf{Keywords:} diversification analysis, MEDUSA, BiSSE, speciation
rate, insect-hostplant dynamics

\section*{Introduction}

Hostplant shifts have been invoked to be responsible for a great part of
the biodiversity of herbivorous insects \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{1}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{mitter1988}
}%DIFAUXCMD
}\DIFaddend . The study of the
evolution of hostplant use has spawned several theories explaining the
evolutionary interactions between plants and insects \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{2}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{nyman2012}
}%DIFAUXCMD
}\DIFaddend . The
``escape-and-radiate'' hypothesis \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{3}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{ehrlich1964}
}%DIFAUXCMD
}\DIFaddend , the 
``oscillation hypothesis''
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{4,5}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{janz2011,nylin2014}
}%DIFAUXCMD
}\DIFaddend or ``diffuse cospeciation'' \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{2}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{nyman2012}
}%DIFAUXCMD
and the
``musical chairs hypothesis'' \mbox{%DIFAUXCMD
\cite{hardy2014}
}%DIFAUXCMD
}\DIFaddend .

The butterfly family Nymphalidae has been an important taxon for
developing some of the mentioned hypotheses. Nymphalidae contains around
6000 species \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{6}%DIFDELCMD < {]}%%%
\DIFdel{,
and is the largest family within the true
butterflies}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{van_nieukerken2011}
}%DIFAUXCMD
,
and several members are considered model organisms
in evolutionary biology \mbox{%DIFAUXCMD
\cite{joron2006,willmott2006,brakefield2009}
}%DIFAUXCMD
}\DIFaddend .
The family most likely originated
around 94 MYA in the mid Cretaceous. Diversification of the group began
in the Late Cretaceous and most major radiations (current subfamilies)
appeared shortly after the Cretaceous-Paleogene (K-Pg) boundary
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{7}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{heikkila2012}
}%DIFAUXCMD
}\DIFaddend . Several studies have used time-calibrated phylogenies and
diversification models to reconstruct the evolutionary history of the
group to identify patterns of accelerated or decelerated diversification
of some Nymphalidae clades
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{7--10}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{heikkila2012,elias2009,fordyce2010,wahlberg2009}
}%DIFAUXCMD
}\DIFaddend .
For example, it has been
suggested that climate change in the Oligocene and the subsequent
diversification of grasses has led to diversification of the subfamily
Satyrinae \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{11}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{pena2008}
}%DIFAUXCMD
}\DIFaddend due to the abundance of grasses over extensive
geographic areas (``resource abundance-dependent diversity dynamics''
hypothesis). \DIFdelbegin \DIFdel{Reference 9 }\DIFdelend \DIFaddbegin \DIFadd{Fordyce (2010) \mbox{%DIFAUXCMD
\cite{fordyce2010}
}%DIFAUXCMD
}\DIFaddend found increased net diversification
rates in some Nymphalidae lineages after a major hostplant shift, which
appears to be in agreement with \DIFdelbegin \DIFdel{Reference 3
}\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend ``escape-and-radiate'' model of
diversification \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{ehrlich1964}
}%DIFAUXCMD
}\DIFaddend .

Although it has been suggested that part of the great diversity of
Nymphalidae butterflies is a result of hostplant-insect dynamics, it is
necessary to use modern techniques to investigate whether the
diversification patterns of Nymphalidae are in agreement with the
theoretical predictions. It is necessary to test whether the overall
diversification pattern of Nymphalidae is congruent with events of
sudden diversification bursts due to hostplant shift \DIFdelbegin \DIFdel{, climatic events
or
shifts to closely related hostplants }%DIFDELCMD < {[}%%%
\DIFdel{5,12}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{or climatic events
\mbox{%DIFAUXCMD
\cite{nylin2014,ferrer2013}
}%DIFAUXCMD
}\DIFaddend .

In this study, we used a time-calibrated genus-level phylogenetic
hypothesis for Nymphalidae butterflies \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{10}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend to
investigate patterns
of diversification. We applied \DIFdelbegin \DIFdel{MEDUSA }%DIFDELCMD < {[}%%%
\DIFdel{13,14}%DIFDELCMD < {]}%%%
\DIFdel{, a recently developed
statistical method
}\DIFdelend \DIFaddbegin \DIFadd{the statistical method
MEDUSA \mbox{%DIFAUXCMD
\cite{alfaro2009,harmon2011}
}%DIFAUXCMD
}\DIFaddend ,
to study the diversification pattern of Nymphalidae
butterflies. MEDUSA fits likelihood models of diversification into a
time-calibrated tree and tests whether allowing increases or decreases
in speciation and extinction rates within the tree produces better fit
of the models. MEDUSA is able to take into account unsampled extant
species diversity during model fitting and it is normally applied to the
maximum clade credibility phylogenetic tree. Particularly, we wanted to
study the effects of phylogenetic uncertainty and by using the extended
MEDUSA method called MultiMEDUSA \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{13}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{alfaro2009}
}%DIFAUXCMD
}\DIFaddend . We also tested whether
hostplant association dynamics can explain the diversification patterns
of component Nymphalidae lineages by testing whether character states of
hostplant use affected the diversification pattern of those lineages
employing the method BiSSE as implemented in the R package
\texttt{diversitree} \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{15}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fitzjohn2012}
}%DIFAUXCMD
}\DIFaddend .

\section*{\DIFaddbegin \DIFadd{Materials and }\DIFaddend Methods}

\subsection*{Data}

For analyses, we used the phylogenetic trees from the study of \DIFdelbegin \DIFdel{Reference
10 }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg
et al. (2009) \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend that were generated using DNA sequence data from
10 gene regions for 398 of the 540 valid genera in Nymphalidae. We
employed \DIFdelbegin \DIFdel{Reference 10
}\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend maximum clade credibility tree (MCC tree) (Fig. 1) as well
as a random sample of 1000 trees from their BEAST run after burnin
\DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend . Their original BEAST run was for 40 million generations. We
used a burnin of 25 million generations and took a random sample of 1000
trees using Burntrees v.0.1.9 \url{http://www.abc.se/~nylander/} (supp.
mat. 5) in order to correct for phylogenetic uncertainty when performing
the diversification analyses.

Species richness data for Nymphalidae genera were compiled from several
sources including the specialist-curated lists on
\url{http://tolweb.org}, \DIFdelbegin \DIFdel{Reference 16 }\DIFdelend \DIFaddbegin \DIFadd{Lamas (2004) \mbox{%DIFAUXCMD
\cite{lamas2004}
}%DIFAUXCMD
}\DIFaddend and curated lists of
Global Butterfly Names project (\url{http://www.ucl.ac.uk/taxome/gbn/}).
We assigned the species numbers of genera not included in the phylogeny
to the closest related genus that was included in \DIFdelbegin \DIFdel{Reference 10 }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009)
\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend study according to available phylogenetic studies 
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{17--28}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{matos2013,brower2010,kodandaramaiah2010,kodandaramaiah2010a,ortiz2013,desilva2010,freitas2004,pena2006,penz1999,silva2008,pena2011,pena2010}
}%DIFAUXCMD
}\DIFaddend ,
taxonomical classification and morphological resemblance when no
phylogenies were available (supp. mat. 3).

Hostplant data for Nymphalidae species were compiled from several
sources including \DIFdelbegin \DIFdel{Reference 29}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{ackery1988}
}%DIFAUXCMD
}\DIFaddend , HOSTS database (\url{http://bit.ly/YI7nwW}),
\DIFdelbegin \DIFdel{Reference 30, Reference 31, Reference 32
}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{dyer2002,beccaloni2008,janzen2009}
}%DIFAUXCMD
}\DIFaddend and others (supp. mat. 15--16)
for a total of 6586
hostplant records, including 428 Nymphalidae genera and 143 plant
families and 1070 plant genera. It was not possible to find any
hostplant data for 35 butterfly genera.

\subsection*{Analyses of Diversification}

We used the statistical software R version 3.0.1 \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{33}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{r2013}
}%DIFAUXCMD
}\DIFaddend in combination
with the APE \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{34}%DIFDELCMD < {]}%%%
\DIFdel{, GEIGER }%DIFDELCMD < {[}%%%
\DIFdel{35}%DIFDELCMD < {]}%%%
\DIFdel{, MEDUSA 
}%DIFDELCMD < {[}%%%
\DIFdel{13}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{popescu2012}
}%DIFAUXCMD
, GEIGER \mbox{%DIFAUXCMD
\cite{harmon2008}
}%DIFAUXCMD
, MEDUSA 
\mbox{%DIFAUXCMD
\cite{alfaro2009}
}%DIFAUXCMD
}\DIFaddend and
\texttt{diversitree} \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{15}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fitzjohn2012}
}%DIFAUXCMD
}\DIFaddend packages along with our own scripts to
perform the analyses (included as supplementary materials 2, 6, 10). All
analyses were run on the 1000 random trees from \DIFdelbegin \DIFdel{Reference 10}\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009)
\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend , on the maximum clade credibility tree derived from these and
the MCC tree \DIFdelbegin \DIFdel{from
Reference 10.
}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
.
}

\DIFadd{We analyzed our MCC tree in the Bayesian software BAMM in order to contrast
the results
found by MEDUSA and MultiMEDUSA.
BAMM (Bayesian analysis of macroevolutionary mixtures)
\mbox{%DIFAUXCMD
\cite{rabosky2014}
}%DIFAUXCMD
is a
software that fits several models of speciation and extinction. The best model
can be selected by comparison of Bayes Factors.
}\DIFaddend 

\subsection*{Detecting diversification shifts on phylogenetic trees}

Patterns of diversification in Nymphalidae were analyzed by using MEDUSA
version 093.4.33 \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{13}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{alfaro2009}
}%DIFAUXCMD
}\DIFaddend on the maximum clade credibility tree from
\DIFdelbegin \DIFdel{Reference 10. }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009) \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
. }\DIFaddend MEDUSA fits alternative birth-death
likelihood models to a phylogenetic tree in order to estimate changes in
net diversification rates along branches. MEDUSA estimates likelihood
and AIC scores for the simplest birth-death model, with two parameters,
the rates \texttt{r}: net diversification \texttt{=} speciation
(\texttt{b}) \texttt{-} extinction (\texttt{d}) and \DIFdelbegin \DIFdel{\(\varepsilon\)}\DIFdelend \DIFaddbegin \DIFadd{($\varepsilon$}\DIFaddend :
relative extinction \texttt{= d/b}. The AIC scores of the two-parameter
model are then compared with incrementally more complex models until the
addition of parameters do not improve the AIC scores beyond a cutoff
value. MEDUSA finds the likelihood of the models after taking into
account branch lengths and number of species per lineage 
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{13}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{alfaro2009}
}%DIFAUXCMD
}\DIFaddend . Most
studies using MEDUSA only run the method on a single tree, usually the
maximum clade credibility tree, which makes the assumption that this
tree is correct. We wanted to study the effects of phylogenetic
uncertainty on estimation of net diversification rate shifts and
therefore selected 1000 random genus-level trees from the posterior
distribution \DIFaddbegin \DIFadd{of the Bayesian run from Wahlberg et al., 2009 \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend (MultiMEDUSA, supp. mat. 5). We calculated a new MCC tree from the
selection of trees, ran MEDUSA and MultiMEDUSA on the selection of 1000
trees and summarized the estimated changes in net diversification rates
for nodes across all trees. Patterns of change in net diversification
rates are considered significant if they are found at the same node in
at least 90\% of the trees. We also expected to find similar values of
net diversification (\texttt{r = b - d}) and relative extinction rate
(\DIFdelbegin \DIFdel{\(\varepsilon\)
}\DIFdelend \DIFaddbegin \DIFadd{$\varepsilon$ }\DIFaddend \texttt{= d/b}) values across the 1000 trees for the
nodes where changes in diversification tempo occurs. We used the AICc
threshold of 7.8 units, as estimated by MEDUSA, as the limit for a
significantly better fit to select among increasingly complex
alternative models.

\subsection*{Estimation of trait-dependent speciation
rates}

As the MEDUSA and MultiMEDUSA approaches estimated an increase in net
diversification in the clade Ithomiini, we tested whether this pattern
can be explained by \DIFdelbegin \DIFdel{increase in }\DIFdelend \DIFaddbegin \DIFadd{an increase in the }\DIFaddend birth-rate due to hostplant use
and performed analyses \DIFdelbegin \DIFdel{with }\DIFdelend \DIFaddbegin \DIFadd{using }\DIFaddend the ``binary state speciation and
extinction'' \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{36}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{maddison2007}
}%DIFAUXCMD
}\DIFaddend Bayesian approach as implemented in the R package
\texttt{diversitree} \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{15}%DIFDELCMD < {]}%%%
\DIFdel{. MuSSE }%DIFDELCMD < {[}%%%
\DIFdel{15}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fitzjohn2012}
}%DIFAUXCMD
. The Markov Chain Monte Carlo algorithm
was run for 10000 generations discarding the first 7500 as burnin. The
Multiple State Speciation and Extinction approach (MuSSE) 
\mbox{%DIFAUXCMD
\cite{fitzjohn2012}
}%DIFAUXCMD
}\DIFaddend is
designed to examine the joint effects of two or more traits on
speciation. Because most of Nymphalidae butterflies are restricted to
use one plant family as hostplant \DIFaddbegin \DIFadd{(232 genera use only one plant family
versus 176 that use more than one; based on our data in supp. mat. 15)}\DIFaddend ,
the character states can be coded as presence/absence, for which the
BiSSE analysis is better suited. BiSSE was designed to test whether a
binary character state has had any effect on increased net
diversification rate for a clade \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{36}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{maddison2007}
}%DIFAUXCMD
}\DIFaddend . We used our compiled data of
hostplant use to produce a binary dataset for the character ``feeding on
the plant family Solanaceae'' with two states (absence = 0; presence =
1) (supp. mat. 17)\DIFdelbegin \DIFdel{, which is the main }\DIFdelend \DIFaddbegin \DIFadd{. The family Solanaceae include the }\DIFaddend hostplants of the
diverse Ithomiini butterflies and closest relatives \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006}
}%DIFAUXCMD
}\DIFaddend . Other
hostplant shifts were not tested for effect on net diversification rates
due to the low support for diversification shifts found in the MEDUSA
analyses. We analyzed the data using BiSSE employing the Markov Chain
Monte Carlo algorithm on the maximum clade credibility tree, taking into
account missing taxa by using the parameter ``sampling factor''
(\texttt{sampling.f}) in \texttt{diversitree}. We also used constrained
analyses forcing no effect of hostplant use on diversification and used
likelihood ratio tests to find out whether the hypothesis of effect on
diversification has a significantly better likelihood than the null
hypothesis (no effect). The analyses were run across a sample of 250
trees from the \DIFaddbegin \DIFadd{selected 1000 random trees from the }\DIFaddend posterior
distribution. The records of \emph{Vanessa} and \emph{Hypanartia}
feeding on Solanaceae \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{30,38}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{beccaloni2008,scott1986}
}%DIFAUXCMD
}\DIFaddend might be incorrect as it
is unlikely
that these species can be feeding on this plant family. We tested
whether coding these two genera as not feeding on Solanaceae affected
our results. \DIFaddbegin \DIFadd{We expected that coding these two genera as either absence
or presence would not have a significant effect on our results. It has
been shown that BiSSE performs poorly under certain conditions 
\mbox{%DIFAUXCMD
\cite{davis2013}
}%DIFAUXCMD
.
However, our data has adequate number of taxa under analysis (more than
300 tips), adequate speciation bias (between 1.5x and 2.0x), character
state bias (around 8x) and extinction bias (around 4x) for the analysis
of Solanaceae hostplants. Thus, BiSSE is expected to produce robust
results \mbox{%DIFAUXCMD
\cite{davis2013}
}%DIFAUXCMD
.
}\DIFaddend 

\section*{Results}

\subsection*{Detecting diversification shifts on the maximum clade
credibility tree}

The MEDUSA analysis on the MCC tree in combination with richness data
estimated 18 \DIFaddbegin \DIFadd{significant }\DIFaddend changes in the tempo of diversification in Nymphalidae
history (Fig. 1; Table 1). The corrected AICc acceptance threshold for
adding subsequent piecewise birth-death processes to the overall model
was set to 7.8 units, as prescribed by MEDUSA. In all MEDUSA analyses,
the maximum number of inferred diversification splits in all trees was
26. The background net diversification rate for Nymphalidae was
estimated as \texttt{r = 0.092} lineages per Million of years and the
AICc score for the best fit model was \texttt{5090.5} (Table 1). MEDUSA
also estimated that the basic constant \DIFdelbegin \DIFdel{birth--death }\DIFdelend \DIFaddbegin \DIFadd{birth-death }\DIFaddend model was not a
better explanation for our data (AICc \texttt{= 5449.3}).

Some of the 18 changes in diversification correspond to rate increases
in very species-rich genera: \emph{Ypthima} (\texttt{r = 0.311}),
\emph{Charaxes} (\texttt{r = 0.291}), \emph{Callicore} +
\emph{Diaethria} (\texttt{r = 0.220}), \emph{Pedaliodes}
(\texttt{r = 0.196}) and \emph{Taenaris} (\texttt{r = 0.234}). We found
rate increases for other clades as well such as: Mycalesina
(\texttt{r = 0.191}), Oleriina + Ithomiina + Napeogenina + Dircennina +
Godyridina (\texttt{r = 0.187}), \DIFdelbegin \DIFdel{Satyrini }\DIFdelend \DIFaddbegin \DIFadd{Satyrinae }\DIFaddend (\texttt{r = 0.116}),
Phyciodina in part (\texttt{r = 0.241}) and Satyrina
(\texttt{r = 0.221}), \emph{Coenonympha} (\texttt{r = 0.209}),
\emph{Caeruleuptychia} + \emph{Magneuptychia} (\texttt{r = 0.312}) and
\emph{Taenaris} (\texttt{r = 0.312}). We also found decreases in net
diversification rates for Limenitidinae + Heliconiinae
(\texttt{r = 0.0541}), part of Danaini (\texttt{r = 0.0423}),
Pseudergolinae (\texttt{r = 0.024}) and Coenonymphina
(\texttt{r = 0.065}) (Table 1).

\DIFaddbegin \DIFadd{We contrasted these results with those obtained from analyzing the data in
the software BAMM \mbox{%DIFAUXCMD
\cite{rabosky2014}
}%DIFAUXCMD
. We found 
that the best model (best configuration) estimated 2 diversification shifts in
our MCC tree (posterior probability = 0.79) for the Ithomiini
and Satyrini (Fig. S1). Bayes factor (BF) values strongly 
favored this model in comparison with a constant rate model (no diversification
shifts, BF $>$ 1,884).
}

\DIFaddend \subsection*{Phylogenetic uncertainty in the MultiMEDUSA
approach}

We \DIFdelbegin \DIFdel{tested }\DIFdelend \DIFaddbegin \DIFadd{used }\DIFaddend MEDUSA to find out whether taking into account the phylogenetic
signal from \DIFdelbegin \DIFdel{a }\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend random sample of 1000 trees from the posterior
distribution can return similar estimates of diversification to the
values obtained from the MCC tree. We \DIFdelbegin \DIFdel{run MultiMEDUSA on a }\DIFdelend \DIFaddbegin \DIFadd{ran MultiMEDUSA on the }\DIFaddend random
sample of 1000 trees (supp. mat. 5) from the posterior distribution and
compared the results with a MEDUSA analysis on the MCC tree derived from
this sample (supp. mat. 9).

We found that the analysis by MultiMEDUSA on the 1000 trees estimated
lower median net diversification rates for the diversification shifts
found by MEDUSA on the MCC tree \DIFdelbegin \DIFdel{from Reference 10 }\DIFdelend \DIFaddbegin \DIFadd{derived from the random sample of trees
}\DIFaddend (Table 2). Although the
diversification pattern found by MEDUSA and MultiMEDUSA was the same,
the latter consistently estimated lower rates. \DIFdelbegin \DIFdel{Thus}\DIFdelend \DIFaddbegin \DIFadd{Furthermore}\DIFaddend , the shifts
recovered with low net diversification rate on the MCC were recovered
with negative net diversification rate by MultiMEDUSA. The background
diversification and all shifts found by MEDUSA on the 1000 trees are
provided as an R object in supp. mat. \DIFdelbegin \DIFdel{7).
}\DIFdelend \DIFaddbegin \DIFadd{7.
}\DIFaddend 

We also compared the results from MultiMEDUSA (derived from the sample
of 1000 trees) with the splits found by MEDUSA on the MCC tree derived
from this random sample. In the summary statistics, MultiMEDUSA reports
the frequency of the diversification shifts found in the trees
(parameter \texttt{sum.prop}). Thus, if a node is found in only half of
the 1000 trees, but the phylogenetic signal was strong enough to be
picked up by MEDUSA and a node shift was found most of the time, then
the \texttt{sum.prop} should be close to 1. For example\DIFdelbegin \DIFdel{the node 625
consists of the clade }\DIFdelend \DIFaddbegin \DIFadd{, the
}\DIFaddend \texttt{Charaxes} + \texttt{Polyura} \DIFdelbegin \DIFdel{and it }\DIFdelend \DIFaddbegin \DIFadd{clade }\DIFaddend was found in only 256 trees,
however MultiMEDUSA was \DIFdelbegin \DIFdel{consistenly }\DIFdelend \DIFaddbegin \DIFadd{consistently }\DIFaddend able to find a diversification
shift for that node and the \texttt{sum.prop} value is 0.996.

\DIFdelbegin \DIFdel{We found different rates for diversification shifts even for the nodes
that appear in }\DIFdelend \DIFaddbegin \DIFadd{For the diversification shifts found in both the MCC tree and }\DIFaddend most of
the \DIFdelbegin \DIFdel{sample }\DIFdelend \DIFaddbegin \DIFadd{samples }\DIFaddend of 1000 trees (\DIFaddbegin \DIFadd{frequency more than 90}\%\DIFadd{; }\DIFaddend Table 2)\DIFdelbegin \DIFdel{as many of the
nodes also found in }\DIFdelend \DIFaddbegin \DIFadd{, the
MultiMEDUSA approach recovered different rates of diversification than
those found when }\DIFaddend the MCC tree \DIFdelbegin \DIFdel{had a probability of being recovered
higher than 0.90}\DIFdelend \DIFaddbegin \DIFadd{alone}\DIFaddend .

There were four diversification shifts found \DIFdelbegin \DIFdel{with more than 90}%DIFDELCMD < \% %%%
\DIFdel{of
probability }\DIFdelend in the trees from the
random sample (Table 2): (i) net
diversification rate increase in the genus \emph{Ypthima} \DIFaddbegin \DIFadd{($r = 0.22$)}\DIFaddend ;
(ii) net diversification rate increase in the genus \emph{Charaxes}
\DIFaddbegin \DIFadd{($r = 0.21$)}\DIFaddend ; (iii) rate increase in Ithomiini subtribes Oleriina +
Ithomiina + Napeogenina + Dircennina + Godyridina \DIFaddbegin \DIFadd{($r = 0.10$)}\DIFaddend ;
and (iv) rate increase in \emph{Callicore} + \emph{Diaethria} \DIFaddbegin \DIFadd{($r = 0.135$)}\DIFaddend .

MultiMEDUSA provided mean and standard deviation statistics for the
diversification values \DIFdelbegin \DIFdel{found }\DIFdelend on the shifts on the 1000 trees (supp. mat.
13--14), and found that some of the changes in net diversification rate
values had great variation across the posterior distribution of trees. A
boxplot of the net diversification rate values estimated for the clades
that appear in the MCC tree shows that some shifts are estimated as
increased or slowed diversification pace depending on the tree used for
analysis (Fig. 2). This variation is especially wide for the clade
formed by the genera \emph{Magneuptychia} and \emph{Caeruleuptychia}
because MEDUSA estimated diversification values from six times the
background net diversification rate (\texttt{r = 0.5234}) to almost zero
(\texttt{r = 1.22e-01}). The rates for \emph{Taenaris} were between 0.14
and 0.44 (mean value 0.25). Similar degrees of variation were found in
the nodes for \emph{Ypthima}, \emph{Charaxes} and \emph{Coenonympha}
(Fig. 2). The net diversification rates estimates for the clades
(Oleriina + Ithomiina + Napeogenina + Dircennina + Godyridina),
Limenitidinae + Heliconiinae and Pseudergolinae are relatively
consistent across the 1000 trees (Fig. 2).

It is also evident that not all the diversification shifts estimated on
the MCC tree are consistently recovered in most of the 1000 trees. Some
of the splits in the MCC tree are recovered in very few trees, for
example the split for the clade \DIFdelbegin \DIFdel{Satyrini (Euptychiina + Pronophilina +
Satyrina + Maniolina) }\DIFdelend \DIFaddbegin \DIFadd{Satyrinae }\DIFaddend is recovered with a probability
of \DIFdelbegin \DIFdel{18}%DIFDELCMD < \% %%%
\DIFdelend \DIFaddbegin \DIFadd{0.18 }\DIFaddend (Fig. 3).

\subsection*{Estimation of trait-dependent speciation
rates}

The MEDUSA analyses\DIFaddbegin \DIFadd{, }\DIFaddend taking into account phylogenetic uncertainty\DIFaddbegin \DIFadd{,
}\DIFaddend estimated a net diversification rate increase in part of the clade
Ithomiini across more than 95\% of the trees. Our BiSSE analysis found a
positive effect of the character state ``feeding on Solanaceae'' on the
net net diversification rate on part of Ithomiini (Oleriina + Ithomiina
+ Napeogenina + Dircennina + Godyridina) (Fig. 4). The \DIFdelbegin \DIFdel{Markov Chain
Monte Carlo algorithm was run for 10000 generations discarding the first
7500 as burnin. The }\DIFdelend estimated mean
net diversification rate for taxa that do not feed on Solanaceae was
\texttt{r = 0.11} while the net diversification rate for the Solanaceae
feeders was \texttt{r = 0.16} (see Fig. \DIFdelbegin \DIFdel{S1 }\DIFdelend \DIFaddbegin \DIFadd{S2 }\DIFaddend for a boxplot of speciation
and extinction values for the 95\% credibility intervals). The same
analysis considering \emph{Vanessa} and \emph{Hypanartia} as
non-Solanaceae feeders due to dubious records produced the same pattern
and net diversification rates (Fig. \DIFdelbegin \DIFdel{S2}\DIFdelend \DIFaddbegin \DIFadd{S3}\DIFaddend ). Therefore, the rest of the
analyses were performed assuming these two genera as Solanaceae feeders.
We constrained the BiSSE likelihood model to force equal rates of
speciation for both character states in order to test whether the model
of different speciation rates is a significantly better explanation for
the data. A likelihood ratio test found that the model for increased net
diversification rate for nymphalids feeding on Solanaceae is a
significantly better explanation than this character state having no
effect on diversification (\DIFdelbegin \texttt{\DIFdel{p \textless{} 0.001}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{\(\chi^2 = 12.3; 1 df; p < 0.001\)}\DIFaddend ) (Table 3;
character states available in supp. mat. 17, code in supp. mat. 18, and
mcmc run in supp. mat. 19). We combined the post-burnin mcmc generations
from running BiSSE on 250 trees from the \DIFdelbegin \DIFdel{posterior distribution }\DIFdelend \DIFaddbegin \DIFadd{random sample of 1000 trees }\DIFaddend and
found the same pattern as the BiSSE analysis on the maximum clade
credibility tree (combined mcmc run in supp. mat. 20; profiles plot of
speciation rates in Fig. \DIFdelbegin \DIFdel{S3}\DIFdelend \DIFaddbegin \DIFadd{S4}\DIFaddend ; boxplot of 95\% credibility intervals in
Fig. \DIFdelbegin \DIFdel{S4}\DIFdelend \DIFaddbegin \DIFadd{S5}\DIFaddend ). A BiSSE analysis to test whether the trait ``feeding on
Apocynaceae'' had any effect on increased net diversification rates
found similar speciation rates for lineages feeding on Apocynaceae and
other plants (Fig. \DIFdelbegin \DIFdel{S5). It has been shown
that BiSSE performs poorly under certain conditions }%DIFDELCMD < {[}%%%
\DIFdel{39}%DIFDELCMD < {]}%%%
\DIFdel{. However,
our data has adequate number of taxa under analysis (more than 300
tips), adequate speciation bias (between 1.5x and 2.0x), character state
bias (around 8x) and extinction bias (around 4x)for the analysis of
Solanaceae hostplants. Thus, BiSSE is expected to produce robust results
}%DIFDELCMD < {[}%%%
\DIFdel{39}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{S6)}\DIFaddend .

\section*{Discussion}

\subsection*{Effects of phylogenetic uncertainty on the performance of
MEDUSA}

The MEDUSA method has been used to infer changes in net diversification
rates \DIFdelbegin \DIFdel{along }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend a phylogenetic tree. Since its publication \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{13}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{alfaro2009}
}%DIFAUXCMD
}\DIFaddend the results
of using MEDUSA on a single tree, the maximum clade credibility tree,
have been used for generation of hypotheses and discussion
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{7,40,41}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{heikkila2012,litman2011,ryberg2012}
}%DIFAUXCMD
}\DIFaddend . However, \DIFdelbegin \DIFdel{we found that MEDUSA estimated }\DIFdelend different diversification
shifts and different
rates of diversification \DIFaddbegin \DIFadd{are found }\DIFaddend for certain lineages when
phylogenetic uncertainty was taken into account by using MEDUSA on a
random sample of trees from the posterior distribution of a Bayesian
run. We found that some diversification splits, estimated on the
Nymphalidae maximum clade credibility tree, were found with \DIFdelbegin \DIFdel{a
}\DIFdelend very low
probability in \DIFdelbegin \DIFdel{a }\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend random sample of 1000 trees from the posterior
distribution (Fig. 3, Table 2). We also found that, even though \DIFdelbegin \DIFdel{MEDUSA
could estimate }\DIFdelend \DIFaddbegin \DIFadd{the
analyses estimated }\DIFaddend the same diversification splits on two or more trees,
the estimated net diversification rates could vary widely (Fig. 2). For
example, in our Nymphalidae trees, we found that the split for
\emph{Magneuptychia} and \emph{Caeruleuptychia} had a variation from
\texttt{r = 0.5234}, higher than the background net diversification
rate, to almost zero. This means that observed patterns and conclusions
can be completely contradictory depending on tree choice.

In this study, the effect of phylogenetic uncertainty on the inferred
diversification splits by MEDUSA is amplified because some Nymphalidae
taxa appear to be strongly affected by long-branch attraction artifacts
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{27}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{pena2011}
}%DIFAUXCMD
}\DIFaddend . Thus, the Bayesian runs are expected to recover alternative
topologies on the posterior distribution of trees, resulting in low
support and posterior probability values for the nodes. For example,
posterior probability values for clades in \DIFdelbegin \DIFdel{Satyrini }\DIFdelend \DIFaddbegin \DIFadd{Satyrinae }\DIFaddend are very low
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{10}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend . As a result, MEDUSA inferred a net diversification rate
increase for \DIFdelbegin \DIFdel{part of Satyrini }\DIFdelend \DIFaddbegin \DIFadd{the Satyrinae (which includes Satyrini and its sister clade)
}\DIFaddend in the maximum clade credibility tree, but
this was recovered \DIFdelbegin \DIFdel{only in 13}%DIFDELCMD < \% %%%
\DIFdel{of probabilities }\DIFdelend in the MultiMEDUSA analysis \DIFdelbegin \DIFdel{on }\DIFdelend \DIFaddbegin \DIFadd{in only 13}\% \DIFadd{of }\DIFaddend the
random sample of trees.

If there is strong phylogenetic signal for increases or decreases in net
diversification rates for a node, it is expected that these splits would
be inferred by MEDUSA in most of the posterior distribution of trees.
However, weak phylogenetic signal for some nodes can cause some clades
to be absent in some trees and MEDUSA will be unable to estimate any
diversification shift (due to a non-existent node). This is the reason
why MEDUSA estimated net diversification rate splits with \DIFdelbegin \DIFdel{more than 90}%DIFDELCMD < \%
%DIFDELCMD < %%%
\DIFdel{of probabilities }\DIFdelend \DIFaddbegin \DIFadd{a probability
higher than 0.90 }\DIFaddend in the sample of trees for only four splits: the genus
\emph{Charaxes}, the genus \emph{Ypthima}, part of Ithomiini and the
clade \emph{Callicore} + \emph{Diaethria} (Fig. 3), while estimating
splits for other lineages with much lower probability.

The clade Ithomiini and the non-basal danaids are well supported by high
posterior probability values in \DIFdelbegin \DIFdel{Reference 10. }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009) \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
.
}\DIFaddend Therefore our MEDUSA analyses recovered an increase in net
diversification rate with probability higher than \DIFdelbegin \DIFdel{90}%DIFDELCMD < \% %%%
\DIFdelend \DIFaddbegin \DIFadd{0.90 }\DIFaddend in the posterior
distribution of trees (Fig. 3).

\DIFaddbegin \DIFadd{The results from BAMM recovered only two significant shifts in our MCC tree
as this method seems to be much more conservative than the MEDUSA approach.
One of these 
shifts is the Satyrini clade. MEDUSA estimated its parent node as
significant shift in only a few trees from the posterior distribution. 
Thus, it might be important to
take into account phylogenetic uncertainty in BAMM as well.
}

\DIFaddend \subsection*{Hostplant use and diversification in
Nymphalidae}

\DIFdelbegin \paragraph*{\DIFdel{Ithomiini}}
%DIFAUXCMD
\DIFdelend \DIFaddbegin \subsubsection*{\DIFadd{Ithomiini}}
\DIFaddend 

Keith Brown suggested that feeding on Solanaceae was an important event
in the diversification of Ithomiini butterflies \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{42}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{brown1987}
}%DIFAUXCMD
}\DIFaddend . Ithomiini
butterflies are exclusively Neotropical and most species feed on
Solanaceae hostplants during larval stage \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006}
}%DIFAUXCMD
}\DIFaddend .
Optimizations of the
evolution of hostplant use on phylogenies evidence a probable shift from
Apocynaceae to Solanaceae in the ancestor of the tribe 
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37,43}%DIFDELCMD < {]}%%%
\DIFdel{.
Reference 9 }\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006,brower2006}
}%DIFAUXCMD
.
Fordyce (2010) \mbox{%DIFAUXCMD
\cite{fordyce2010}
}%DIFAUXCMD
}\DIFaddend found that the Gamma statistics, a LTT plot of
an Ithomiini phylogeny and the fit of the density-dependent model of
diversification are consistent with a burst of diversification in
Ithomiini following the shift from Apocynaceae to Solanaceae.

We investigated whether the strong signal for an increase in net
diversification rate for Ithomiini (found by MEDUSA) can be explained
due to the use of Solanaceae plants as hosts during larval stage. For
this, we used a Bayesian approach \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{44}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fitzjohn2009}
}%DIFAUXCMD
}\DIFaddend to test whether the trait
``feeding on Solanaceae'' had any effect on the diversification of the
group.

Our BiSSE analysis, extended to take into account missing taxa and
phylogenetic uncertainty, shows a significantly higher net
diversification rate for Ithomiini taxa, which can be attributed to the
trait ``feeding on Solanaceae hostplants'' (Fig. 4). This is in
agreement with \DIFdelbegin \DIFdel{the findings of Reference 9 }\DIFdelend \DIFaddbegin \DIFadd{previous findings }\DIFaddend using other statistical methods
\DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fordyce2010}
}%DIFAUXCMD
}\DIFaddend . Due to the fact that Ithomiini are virtually the only
nymphalids using Solanaceae as hostplants, it is possible that the trait
responsible for a higher diversification of Ithomiini might not be the
hostplant character. As noted by \DIFdelbegin \DIFdel{Reference 36}\DIFdelend \DIFaddbegin \DIFadd{Maddison et al. (2007) \mbox{%DIFAUXCMD
\cite{maddison2007}
}%DIFAUXCMD
}\DIFaddend ,
the responsible trait might be a codistributed character such as a trait
related to the ability to digest secondary metabolites.

Solanaceae plants contain chemical compounds and it has been suggested
that the high diversity of Ithomiini is consistent with the
``escape-and-radiate scenario'' due to a shift onto Solanaceae 
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{9}%DIFDELCMD < {]}
%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{fordyce2010}
}%DIFAUXCMD
}\DIFaddend and radiation scenarios among chemically different lineages of
Solanaceae plants \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37,42}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006,brown1987}
}%DIFAUXCMD
}\DIFaddend .
According to this theory, the shift from
Apocynaceae to Solanaceae allowed Ithomiini to invade newly available
resources due to a possible key innovation that allowed them cope with
secondary metabolites of the new hosts. Additional studies are needed to
identify the actual enzymes that Ithomiini species might be using for
detoxification of ingested food as they have been found in other
butterfly groups \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{45}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wheat2007}
}%DIFAUXCMD
}\DIFaddend .

The increase in diversification inferred by MEDUSA \DIFdelbegin \DIFdel{ocurred }\DIFdelend \DIFaddbegin \DIFadd{occurred }\DIFaddend after the
probable shift from Apocynaceae to Solanaceae, as the Solanaceae feeders
in the subtribes Melinaeina and Mechanitina are not included in the
diversification shift (shift number 5 in Fig. 1). The apparent
conflicting results from MEDUSA and BiSSE can be explained by the low
species-richness of the subtribes Melinaeina and Mechanitina compared to
the other subtribes included in the shift (52 versus 272 species). It
can be that MEDUSA is more conservative than BiSSE and is not including
Melinaeina and Mechanitina in the shift due to low species numbers.

Although the Solanaceae genera used by the Ithomiini clades are well
known \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006}
}%DIFAUXCMD
}\DIFaddend , we do not have any understanding on the physiological
routes involved in the detoxification of Solanaceae compounds by the
several lineages of Ithomiini. We can speculate that older lineages
exploiting a novel toxic resource \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{10,37}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006,wahlberg2009}
}%DIFAUXCMD
}\DIFaddend might not be too efficient
in metabolizing plant toxins and that younger lineages are able to deal
with toxins more efficiently, so that host switching events within
Solanaceae are possible, which can lead to higher diversification.
Studies in \emph{Papilio} species have reported that detoxification
enzymes can become more efficient in metabolizing toxins than ancestral
configurations of the proteins, providing more opportunities for
hostplant switch \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{46}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{li2003}
}%DIFAUXCMD
}\DIFaddend . This might be the reason why the basal
Ithomiini subtribes Melinaeina and Mechanitina are so species-poor and
restricted to few Solanaceae hosts \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006}
}%DIFAUXCMD
}\DIFaddend , while recent subtribes are
species-rich and have expanded their host range into several Solanaceae
lineages \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{37}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{willmott2006}
}%DIFAUXCMD
}\DIFaddend . It might be that the switch to feeding in Solanaceae
was an important event in the evolutionary history of Ithomiini, but the
actual radiation occurred after critical physiological changes (a
probable key innovation) allowed efficient detoxification of Solanaceae
toxins.

The diffuse cospeciation hypothesis predicts almost identical ages of
insects and their hostplants, while the ``resource abundance-dependent
diversity'' and the ``escape-and-radiate'' hypotheses state that insects
diversify after their hostplants \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{2--4}%DIFDELCMD < {]}%%%
\DIFdel{.
Reference 45 }\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{nyman2012,ehrlich1964,janz2011}
}%DIFAUXCMD
.
Wheat et al. (2007)
\mbox{%DIFAUXCMD
\cite{wheat2007}
}%DIFAUXCMD
}\DIFaddend found strong evidence for a model of speciation congruent with
Ehrlich and Raven's hypothesis in Pieridae butterflies due to, in
addition to the identification of a key innovation, a burst of
diversification in glucosinolate-feeding taxa shortly afterwards (with a
lag of \DIFdelbegin \DIFdel{\textasciitilde{}}\DIFdelend \DIFaddbegin \DIFadd{circa }\DIFaddend 10 MY). According to a recent dated phylogeny of
the Angiosperms \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{47}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{bell2010}
}%DIFAUXCMD
}\DIFaddend , the family Solanaceae split from its sister
group about 59 (\DIFdelbegin \DIFdel{49-68}\DIFdelend \DIFaddbegin \DIFadd{49--68}\DIFaddend ) MYA and diversification started (crown group
age) around 37 (\DIFdelbegin \DIFdel{29-47}\DIFdelend \DIFaddbegin \DIFadd{29--47}\DIFaddend ) MYA. \DIFdelbegin \DIFdel{Reference 10 give the
corresponding ages for Ithomiini as }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009) \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
give the
ages for origin and diversification for Ithomiini at }\DIFaddend 45 (\DIFdelbegin \DIFdel{39-53}\DIFdelend \DIFaddbegin \DIFadd{39--53}\DIFaddend ) and 37
(\DIFdelbegin \DIFdel{32-43}\DIFdelend \DIFaddbegin \DIFadd{32--43}\DIFaddend ) MYA, respectively. Thus, current evidence shows that Solanaceae
and Ithomiini might have diversified around the same time, during the
Late Eocene and Oligocene, and this would be congruent with the diffuse
cospeciation hypothesis.

\DIFdelbegin \DIFdel{The uplift of the Andes was a major tectonic event that underwent higher
activity during the Oligocene }%DIFDELCMD < {[}%%%
\DIFdel{48}%DIFDELCMD < {]}%%%
\DIFdel{. This caused climatic changes in
the region that affected the flora and fauna of the time, which
coincides with the diversification of modern montane plant and animal
taxa }%DIFDELCMD < {[}%%%
\DIFdel{49}%DIFDELCMD < {]} %%%
\DIFdel{including Ithomiini butterflies and Solanaceae hostplants.
Moreover, all Solanaceae clades currently present in New World
originated in South America }%DIFDELCMD < {[}%%%
\DIFdel{50}%DIFDELCMD < {]} %%%
\DIFdel{as well as Ithomiini butterflies
}%DIFDELCMD < {[}%%%
\DIFdel{10}%DIFDELCMD < {]}%%%
\DIFdel{. This suggests a process of ``diffuse cospeciation'' of
Ithomiini and hostplants.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\paragraph{\DIFdel{Danaini}}
%DIFAUXCMD
\addtocounter{paragraph}{-1}%DIFAUXCMD
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \subsubsection*{\DIFadd{Danaini}}
\DIFaddend Our MultiMEDUSA approach \DIFdelbegin \DIFdel{gave }\DIFdelend \DIFaddbegin \DIFadd{showed }\DIFaddend a significant slowdown in net
diversification rate in the subtribe Danaina of the Danini. Both Danaina
and the sister clade Euploeina feed mainly on Apocynaceae and thus a
hostplant shift should not be responsible for the observed slowdown of
diversification in the Danaina. As expected, our BiSSE analysis of
Apocynaceae feeders shows that there is no effect of feeding on this
plant family on the net diversification rates of Nymphalidae lineages.
Many of the Danaina are large, strong fliers, highly migratory and
involved in mimicry rings. Among them is for example the monarch
(\emph{Danaus plexippus}), probably the most well known of all migratory
butterflies. The causes for a lower net diversification rate in the
Danaina remains to be investigated, but their great dispersal power
might be involved in preventing allopatric speciation. It has been found
in highly vagile species in the nymphalid genus \emph{Vanessa} that
dispersal has homogenized populations due to gene flow, as old and
vagile species seem to be genetically homogeneous while younger
widespread species show higher genetic differentiation in their
populations \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{51}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2011}
}%DIFAUXCMD
}\DIFaddend .

\DIFdelbegin \paragraph{\emph{\DIFdel{Charaxes}}%DIFAUXCMD
}
%DIFAUXCMD
\addtocounter{paragraph}{-1}%DIFAUXCMD
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{The genus }\emph{\DIFdel{Charaxes}} %DIFAUXCMD
\DIFdel{contain 193 species distributed in the Old
World with highest diversity in the Afrotropical region. These
butterflies are also very strong fliers, but contrary to Danaina, which
are specialized Apocynaceae feeders, }\emph{\DIFdel{Charaxes}} %DIFAUXCMD
\DIFdel{are known to feed
on at least 28 plant families in 18 orders }%DIFDELCMD < {[}%%%
\DIFdel{29}%DIFDELCMD < {]} %%%
\DIFdel{and some species
appear to be polyphagous }%DIFDELCMD < {[}%%%
\DIFdel{52}%DIFDELCMD < {]}%%%
\DIFdel{. Reference 53 showed that most of the
diversification of the genus occurred from the late Oligocene to the
Miocene when there were drastic global climatic fluctuations, indicating
that the diversity mainly is driven by climate change. Reference 52
found that climatic changes during the Pliocene, Pleistocene as well as
dispersal and vicariance might have been responsible for the high
diversification of the genus.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\paragraph{\DIFdel{Satyrini}}%DIFAUXCMD
\addtocounter{paragraph}{-1}%DIFAUXCMD
%DIFDELCMD < \label{satyrini}
%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \subsubsection*{\DIFadd{Satyrinae}}
\DIFaddend 

\DIFdelbegin \DIFdel{The diverse tribe Satyrini }\DIFdelend \DIFaddbegin \DIFadd{Lineages in the diverse family Satyrinae }\DIFaddend radiated simultaneously with the
radiation of
their main hostplant, grasses, during the climatic cooling in the
Oligocene \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{11}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{pena2008}
}%DIFAUXCMD
}\DIFaddend . Thus, it is somewhat surprising that part of
\DIFdelbegin \DIFdel{Satyrini (the subtribes Euptychiina, Satyrina and Pronophilina) }\DIFdelend \DIFaddbegin \DIFadd{Satyrinae }\DIFaddend were
found to have accelerated diversification in only 13\% of \DIFdelbegin \DIFdel{probabilities
in }\DIFdelend the trees from
the posterior distribution. Although this can be attributed to low
phylogenetic signal \DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{10}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend , the clade Satyrini is very robust
\DIFdelbegin %DIFDELCMD < {[}%%%
\DIFdel{10}%DIFDELCMD < {]} %%%
\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
}\DIFaddend and MEDUSA failed to identify any significant accelerated net
diversification rate for Satyrini. It appears that the radiation of
Satyrini as a whole was not remarkably fast and therefore not picked up
by MEDUSA\DIFdelbegin \DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{The origin of the tribe Satyrini is not completely clear (originated
either in the Neotropical or Eastern Palaearctic, Oriental and/or
Indo-Australian regions }%DIFDELCMD < {[}%%%
\DIFdel{27}%DIFDELCMD < {]} %%%
\DIFdel{and their radiation involved colonizing
almost all continents starting from their place of origin. For
butterflies, a prerequisite for colonizing new areas is that suitable
hostplants are already present. Therefore, it }\DIFdelend \DIFaddbegin \DIFadd{, although it estimated a diversification shift for Satyrini + its
sister clade. This }\DIFaddend should be expected \DIFdelbegin \DIFdel{that
}\DIFdelend \DIFaddbegin \DIFadd{if }\DIFaddend the diversification of \DIFdelbegin \DIFdel{Satyrinae }\DIFdelend \DIFaddbegin \DIFadd{Satyrini
}\DIFaddend occurred in a stepwise manner, with pulses or bursts of diversification
for certain lineages but unlikely for the tribe Satyrini as a whole.

\DIFaddbegin \section*{\DIFadd{Conclusions}}
\DIFaddend We found that even though MEDUSA estimated several diversification
shifts in the maximum clade credibility tree of Nymphalidae, only a few
of these splits were found in more than 90\% of the trees from the
posterior distribution. In the literature, it is common practice that
conclusions are based on the splits estimated on the maximum clade
credibility tree. However, by using a MultiMEDUSA approach, we found
that \DIFdelbegin \DIFdel{some of this }\DIFdelend \DIFaddbegin \DIFadd{for this Nymphalidae dataset some of these }\DIFaddend splits might be greatly
affected by phylogenetic uncertainty. Moreover, some of these splits can
be recovered either as increases or decreases in net diversification
rate depending on the tree from the posterior distribution that was used
for analysis. This means that contradictory conclusions would be made if
only the maximum clade credibility tree was used for analysis. \DIFaddbegin \DIFadd{We
recommend that all datasets should be analyzed using both approaches,
MEDUSA and MultiMEDUSA, in order to test whether the results are robust
when phylogenetic uncertainty is taken into account.
}

\DIFaddend MEDUSA appears to be sensitive to the number of nodes with high
posterior probability and width of age confidence intervals. For our
data, it would be necessary to obtain a posterior distribution of trees
with no conflicting topology, and very similar estimated ages for nodes
in order to consistently recover most of the diversification splits on
the posterior distribution of trees that were inferred by MEDUSA on the
MCC tree.

Our MultiMEDUSA approach to perform analyses on the posterior
distribution of trees found strong support for an increase in net
diversification rate in the tribe Ithomiini and the genus
\emph{Charaxes}, and for a decrease in net diversification rate in the
subtribe Danaina. Due to phylogenetic uncertainty, we did not obtain
strong support for other diversification splits in Nymphalidae. Our
BiSSE analysis corroborated other studies in that the trait 
``feeding on Solanaceae'', or a codistributed character, was important in the
diversification of Ithomiini butterflies. However, by applying MEDUSA we
found that a critical character in the radiation of the group might have
appeared after the shift from Apocynaceae to Solanaceae. We also found
that the trait ``feeding on Apocynaceae'' is not responsible for the
slowdown of diversification in Danaina. Ithomiini and Solanaceae
diversified near simultaneously, which is in agreement with the diffuse
\DIFdelbegin \DIFdel{co-speciation hypothesis }%DIFDELCMD < {[}%%%
\DIFdel{2,4}%DIFDELCMD < {]}%%%
\DIFdelend \DIFaddbegin \DIFadd{cospeciation hypothesis \mbox{%DIFAUXCMD
\cite{nyman2012,janz2011}
}%DIFAUXCMD
}\DIFaddend .

\DIFdelbegin \subsection*{\DIFdel{Acknowledgments}}%DIFAUXCMD
%DIFDELCMD < \label{acknowledgments}
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \section*{\DIFadd{Acknowledgments}}
\DIFaddend We are thankful to Mark Cornwall for help with the script to extend
MEDUSA to include phylogenetic uncertainty, Niklas Wahlberg for
commenting on the manuscript and giving us the posterior distribution of
trees, Luke Harmon for commenting on the manuscript and anonymous
reviewers for their comments, which greatly improved the manuscript,
Jessica Slove Davidson and Niklas Janz for access to their hostplant
data. The study was supported by a Kone Foundation grant (awarded to
Niklas Wahlberg), Finland (C. \DIFdelbegin \DIFdel{Pea}\DIFdelend \DIFaddbegin \DIFadd{Pe\~na}\DIFaddend ) and the Research Council of Norway
(grant no. 204308 to M. Espeland). We acknowledge CSC--IT Center for
Science Ltd. (Finland) for the allocation of computational resources.

\DIFdelbegin \subsection*{\DIFdel{Figure legends}}%DIFAUXCMD
%DIFDELCMD < \label{figure-legends}
%DIFDELCMD < 

%DIFDELCMD < %%%
\texttt{\DIFdel{Figure 1.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \section*{\DIFadd{Figure legends}}
\begin{description}
 \item {\bf \DIFadd{Figure 1. }\DIFaddend Results of the MEDUSA analysis run on the maximum
clade credibility tree from \DIFdelbegin \DIFdel{Reference 10. }\DIFdelend \DIFaddbegin \DIFadd{Wahlberg et al. (2009)}} \DIFaddend Rate shifts were estimated for the
following nodes (besides the background rate): 2) Limenitidinae +
Heliconiinae, 3) \emph{Ypthima}, 4) \emph{Charaxes}, 5) Ithomiini in
part, 6) \DIFdelbegin \DIFdel{Satyrini}\DIFdelend \DIFaddbegin \DIFadd{Satyrinae}\DIFaddend , 7) Coenonymphina 8) Phyciodina in part, 9) Danaini in
part, 10) \emph{Coenonympha}, 11) \emph{Caeruleuptychia} +
\emph{Magneuptychia}, 12) \emph{Callicore} + \emph{Diaethria}, 13)
Satyrina, 14) \DIFdelbegin \DIFdel{Mycalesinaa}\DIFdelend \DIFaddbegin \DIFadd{Mycalesina}\DIFaddend , 15) \emph{Pedaliodes}, 16) \emph{Dryas} +
\emph{Dryadula}, 17) \emph{Taenaris}, 18) Pseudergolinae. \DIFaddbegin \DIFadd{Circles on
nodes indicate the diversification shift number as found by MEDUSA.
Numbers next to circles indicate the posterior probability values for
such nodes.
}\DIFaddend 

\DIFdelbegin \texttt{\DIFdel{Figure 2.}} %DIFAUXCMD
\DIFdel{Boxplot of the range of diversification values for
tips or clades }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure 2. Diversification rates for taxa }\DIFaddend estimated by MEDUSA on
the \DIFaddbegin \DIFadd{samples of }\DIFaddend 1000 random trees\DIFdelbegin \DIFdel{from the
posterior distribution of the Nymphalidae phylogeny. The tips or clades
shown are those present on the maximum clade credibility tree from
Reference 10.
}\DIFdelend \DIFaddbegin }
\DIFaddend 

\DIFdelbegin \texttt{\DIFdel{Figure 3.}} %DIFAUXCMD
\DIFdel{Results of the MultiMEDUSA analysis on 1000 randomly
sampled trees from the posterior distribution of the Nymphalidae
phylogeny. Bars show the
probability for nodes of being recovered as
significant increases or decreases in net diversification rates by
MultiMEDUSA.
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure 3. Results of MultiMEDUSA analysis showing the
probability of specific nodes being characterized by significant shifts
in diversification rate}}
\DIFaddend 

\DIFdelbegin \texttt{\DIFdel{Figure 4.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure 4. }\DIFaddend BiSSE analysis of diversification of nymphalids due
to feeding on Solanaceae hostplants\DIFdelbegin \DIFdel{. }\DIFdelend \DIFaddbegin } \DIFaddend Speciation and net diversification
rates are significantly higher in Solanaceae feeders (\DIFdelbegin %DIFDELCMD < \$%%%
\DIFdel{lambda}%DIFDELCMD < \$%%%
\DIFdel{1, r1).
}%DIFDELCMD < 

%DIFDELCMD < %%%
\subsection*{\DIFdel{Tables}}%DIFAUXCMD
%DIFDELCMD < \label{tables}
%DIFDELCMD < 

%DIFDELCMD < %%%
\texttt{\DIFdel{Table 1.}} %DIFAUXCMD
\DIFdel{Significant net diversification rate shifts found in
the MEDUSA analysis of the Nymphalid maximum clade credibility tree.
Split.Node = node number, Model = preferred diversification model by
MEDUSA, r }\DIFdelend \DIFaddbegin \DIFadd{speciation rate }\DIFaddend =
\DIFaddbegin \DIFadd{$\lambda1$, }\DIFaddend net diversification rate \DIFdelbegin \DIFdel{, LnLik.part }\DIFdelend = \DIFdelbegin \DIFdel{log likelihood value.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\texttt{\DIFdel{Table 2.}} %DIFAUXCMD
\DIFdel{Differences in rates estimated by MEDUSA on the MCC
tree from Wahlberg and the MultiMEDUSA approach on 1000 random trees
from the posterior distribution.}%DIFDELCMD < 

%DIFDELCMD < %%%
\texttt{\DIFdel{Table 3.}} %DIFAUXCMD
\DIFdel{Likelihood ratio test between the model of increased
diversification of nymphalids feeding on Solanaceae against a model
forcing equal speciation rates (no effect on diversification).}%DIFDELCMD < 

%DIFDELCMD < %%%
\section*{\DIFdel{Supp. mat.}}%DIFAUXCMD
%DIFDELCMD < \label{supp.-mat.}
%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{$r1$.
}\end{description}
\DIFaddend 


\DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 01 genus.nex}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \section*{\DIFadd{Supporting Information Legends}}
\begin{description}
 \item {\bf \DIFadd{Supporting Information S01.nex.}} \DIFaddend MCC Nymphalidae tree from
\DIFdelbegin \DIFdel{Reference 10.
}\DIFdelend \DIFaddbegin {[}\DIFadd{13}{]}
 \DIFaddend 

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 02 run medusa on mct.R}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: Run MEDUSA on MCC tree from Reference 10.
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S02.R}} \DIFadd{R script to run MEDUSA on the
MCC Nymphalidae tree from \mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
. This script removes the outgroup
taxa and loads the richness data for the tree terminals.
}\DIFaddend 

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 03 richness.nex}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S03.csv}} \DIFaddend Species richness for lineages
in Nymphalidae.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 04 MEDUSA results.txt}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S04.txt}} \DIFaddend Results of running MEDUSA on
the MCC tree.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 05 1000 random trees no outgroups.nex}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S05 1000 random trees no outgroups.zip}} \DIFaddend 1000 random trees from \DIFdelbegin \DIFdel{Reference 10.
}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
.
 }\DIFaddend 

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 06 run multimedusa.R}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: Run MultiMEDUSA
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S06.R}}\DIFadd{R script to run a MultiMEDUSA
analysis }\DIFaddend on 1000 random trees from \DIFdelbegin \DIFdel{Reference 10.
}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{wahlberg2009}
}%DIFAUXCMD
.
}\DIFaddend 

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 07 multimedusa on 1000 trees.txt}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S07 multimedusa on 1000 trees.zip}} \DIFaddend Raw results from the MultiMEDUSA run on the random sample of trees from
the posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 08 multimedusa result01.pdf}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: Summary of MulitMEDUSA }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S08.pdf}} \DIFadd{Summary of the MultiMEDUSA
}\DIFaddend analysis on the 1000 trees from the posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 09 mcc tree from random sample of 1000.tree}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S09.tree}} \DIFaddend MCC tree from the 1000
random trees selected from the posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 10 run medusa on mct from 1000 trees.R}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
Run MEDUSA
on }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S10.R}} \DIFadd{R script to run a MEDUSA
analysis on the }\DIFaddend MCC tree from the 1000 random trees selected from the
posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 11 results of MEDUSA run on mcc from 1000 trees.txt}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S11.txt}} \DIFaddend Summary results from a MEDUSA
run on the MCC tree from the 1000 random trees selected from the
posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 12 MEDUSA mcc tree from random sample of 1000 trees.pdf}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S12.pdf}} \DIFaddend Figure for MEDUSA run on MCC
tree from random 1000 trees.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 13 results of Multi-MEDUSA run on 1000 trees.txt}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item{\bf \DIFadd{Supporting Information S13.txt}} \DIFaddend Summary results from a
MultiMEDUSA run on the 1000 random trees selected from the posterior
distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 14 multimedusa result prob nodes.pdf}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S14.pdf}} \DIFaddend Probability of nodes with
estimated rates from a MultiMEDUSA run on the 1000 random trees selected
from the posterior distribution.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 15 host plant data.csv}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S15.csv}} \DIFaddend Hostplants of Nymphalidae
butterflies recorded from the literature.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 16 host plant data references.csv}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{:
}\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S16.csv}} \DIFaddend References for hostplants
data.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 17 hostplant states.csv}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S17.csv}} \DIFaddend Data matrix with character
states for hosplant use.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 18 bisse source.R}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: Rcode }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S18.R}} \DIFadd{R script }\DIFaddend for running the BiSSE
analysis.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 19 bisse mcmc run.csv}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S19.csv}} \DIFaddend Raw results for the BiSSE
analysis.

 \DIFdelbegin \textbf{\texttt{\DIFdel{supp mat 20 combined bisse.csv}}%DIFAUXCMD
}%DIFAUXCMD
\DIFdel{: }\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Supporting Information S20.csv}} \DIFaddend Raw results for the combined
BiSSE analysis.

 \DIFdelbegin \texttt{\DIFdel{Figure S1.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure S1.}} \DIFadd{Best diversification model as estimated by BAMM. Two
diversification shifts were recovered (Ithomiini and Satyrini). The MEDUSA 
analysis found a diversification shifts in Satyrinae (the parent node of 
Satyini).
}

 \item {\bf \DIFadd{Figure S2.}} \DIFaddend Boxplot of speciation (\DIFdelbegin \DIFdel{lambda}\DIFdelend \DIFaddbegin \DIFadd{$\lambda$}\DIFaddend ) and extinction
(\DIFdelbegin \DIFdel{mu}\DIFdelend \DIFaddbegin \DIFadd{$\mu$}\DIFaddend ) values for the 95\% credibility intervals of values estimated
by BiSSE analysis of diversification due to feeding on Solanaceae
plants.

 \DIFdelbegin \texttt{\DIFdel{Figure S2.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure S3.}} \DIFaddend BiSSE analysis of diversification of nymphalids due
to feeding on Solanaceae hostplants assuming \emph{Vanessa} and
\emph{Hypanartia} as non-Solanaceae feeders. The same pattern is
recovered, speciation and net diversification rates are significantly
higher for Solanaceae feeders \DIFdelbegin \DIFdel{(}%DIFDELCMD < \$%%%
\DIFdel{lambda}%DIFDELCMD < \$%%%
\DIFdel{1}\DIFdelend \DIFaddbegin \DIFadd{$\lambda1$}\DIFaddend , r1).

 \DIFdelbegin \texttt{\DIFdel{Figure S3.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure S4.}} \DIFaddend Net diversification rates of nymphalids feeding on
Solanaceae plants as estimated by combining post-burnin runs of BiSSE on
the 1000 trees from the posterior distribution.

 \DIFdelbegin \texttt{\DIFdel{Figure S4.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure S5.}} \DIFaddend Boxplot of speciation and extinction values for the
95\% credibility intervals of values estimated by BiSSE analysis of
diversification due to feeding on Solanaceae plants on the combined
post-burnin runs on 1000 trees from the posterior distribution.

 \DIFdelbegin \texttt{\DIFdel{Figure S5.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \item {\bf \DIFadd{Figure S6.}} \DIFaddend BiSSE analysis of diversification of nymphalids due
to feeding on Apocynaceae hostplants. Speciation and net diversification
rates are similar.
\DIFaddbegin \end{description}
\DIFaddend 



\DIFdelbegin \subsection*{\DIFdel{References}}%DIFAUXCMD
%DIFDELCMD < \label{references}
%DIFDELCMD < %%%
\addcontentsline{toc}{subsection}{\DIFdel{References}}
%DIFAUXCMD
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{1. Mitter C, Farrell B, Wiegmann B (1988) The phylogenetic study of
adaptive zones: Has phytophagy promoted insect diversification? Am Nat
132: 107--128.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1086/284840}{10.1086/284840}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{2. Nyman T, Linder HP, Pea C, Malm T, Wahlberg N (2012) Climate-driven
diversity dynamics in plants and plant-feeding insects. Ecol Lett 15:
889--898.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1461-0248.2012.01782.x}{10.1111/j.1461-0248.2012.01782.x}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{3. Ehrlich PR, Raven PH (1964) Butterflies and plants: A study in
coevolution. Evolution 18: 586--608.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.2307/2406212}{10.2307/2406212}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{4. Janz N (2011) Ehrlich and Raven Revisited: Mechanisms Underlying
Codiversification of Plants and Enemies. Annu Rev Ecol Evol S 42:
71--89.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1146/annurev-ecolsys-102710-145024}{10.1146/annurev-ecolsys-102710-145024}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{5. Nylin S, Slove J, Janz N (2014) Host plant utilization, host range
oscillations and diversification in Nymphalid butterflies: A
phylogenetic investigation. Evolution 68: 105--124.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/evo.12227}{10.1111/evo.12227}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{6. Nieukerken EJ van, Kaila L, Kitching IJ, Kristensen NP, Lees DC, et
al. (2011) Order Lepidoptera. In: Animal biodiversity: An outline of
higher-level classification and survey of taxonomic richness. Zootaxa.
pp. 212--221.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{7. Heikkil M, Kaila L, Mutanen M, Pea C, Wahlberg N (2012) Cretaceous
origin and repeated tertiary diversification of the redefined
butterflies. P Roy Soc B 279: 1093--1099.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rspb.2011.1430}{10.1098/rspb.2011.1430}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{8. Elias M, Joron M, Willmott KR, Silva-Brando KL, Kaiser V, et al.
(2009) Out of the Andes: Patterns of diversification in clearwing
butterflies. Mol Ecol 18: 1716--1729.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1365-294X.2009.04149.x}{10.1111/j.1365-294X.2009.04149.x}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{9. Fordyce JA (2010) Host shifts and evolutionary radiations of
butterflies. P Roy Soc B 277: 3735--3743.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rspb.2010.0211}{10.1098/rspb.2010.0211}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{10. Wahlberg N, Leneveu J, Kodandaramaiah U, Pea C, Nylin S, et al.
(2009) Nymphalid butterflies diversify following near demise at the
Cretaceous/Tertiary boundary. P Roy Soc B 276: 4295--4302.doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rspb.2009.1303}{10.1098/rspb.2009.1303}%%%
\DIFdel{.}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{11. Pea C, Wahlberg N (2008)                                         Prehistorical climate change increased
diversification of a group of butterflies. Biol Lett 4: 274--278.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rsbl.2008.0062}{10.1098/rsbl.2008.0062}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{12. Ferrer-Paris JR, S}%DIFDELCMD < {%%%
\DIFdel{}%DIFDELCMD < }%%%
\DIFdel{nchez-Mercado A, Viloria L, Donaldson J (2013)
Congruence and Diversity of Butterfly-Host Plant Associations at Higher
Taxonomic Levels. PloS one 8: e63570.
doi:}%DIFDELCMD < \href{http://dx.doi.org/0.1371/journal.pone.0063570}{0.1371/journal.pone.0063570}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{13. Alfaro ME, Santini F, Brock C, Alamillo H, Dornburg A, et al. (2009)
Nine exceptional radiations plus high turnover explain species diversity
in jawed vertebrates. P Natl Acad Sci USA 106: 13410--13414.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1073/pnas.0811087106}{10.1073/pnas.0811087106}%%%
\DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{14. Harmon LJ, Rabosky DL, }%DIFDELCMD < {%%%
\DIFdel{FitzJohn}%DIFDELCMD < } %%%
\DIFdel{RG, Brown JW (2011) turboMEDUSA:
MEDUSA: Modeling Evolutionary Diversification Using Stepwise AIC.
}\DIFdelend %DIF >  Use the PLoS provided BiBTeX style
\DIFaddbegin \bibliography{refs}{}
\bibliographystyle{plos2009}
\DIFaddend 



\DIFdelbegin \DIFdel{15. }%DIFDELCMD < {%%%
\DIFdel{FitzJohn}%DIFDELCMD < } %%%
\DIFdel{RG (2012) Diversitree: comparative phylogenetic analyses
of diversification in R. Method Ecol Evol }\DIFdelend %DIF >  Table 1
\DIFaddbegin \section*{\DIFadd{Tables}}
\begin{table}[!h]
\caption{\bf{Significant net diversification rate shifts found in the MEDUSA analysis of the Nymphalid maximum clade credibility tree.}}
\begin{tabular}{lccrrcl}
\DIFaddFL{Shift N$^\circ$ }& \DIFaddFL{Split.Node }& \DIFaddFL{Model }& \DIFaddFL{r          }& \DIFaddFL{LnLik.part }& \DIFaddFL{AICc     }& \DIFaddFL{Taxa                                                       }\\
\DIFaddFL{1               }& \DIFaddFL{399        }& \DIFaddFL{yule  }& \DIFaddFL{0.092459   }& \DIFaddFL{-1055.957  }&          & \DIFaddFL{Nymphalidae (root)                                         }\\
\DIFaddFL{2               }& \DIFaddFL{691        }& \DIFaddFL{bd    }& \DIFaddFL{0.054129   }& \DIFaddFL{-406.3703  }&          & \DIFaddFL{Limenitidinae + Heliconiinae                               }\\
\DIFaddendFL 3               \DIFdelbeginFL \DIFdelFL{: 1084--1092.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.2041-210X.2012.00234.x}{10.1111/j.2041-210X.2012.00234.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{16. Lamas G (2004) Checklist: Part 4A. Hesperioidea-Papilionoidea.
Gainesville.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{17. Matos-Marav PF, Pea C, Willmott KR, Freitas AV, Wahlberg N (2013)
Systematics and evolutionary history of butterflies in the ``Taygetis
clade'' (Nymphalidae: Satyrinae: Euptychiina): Towards a better
understanding of Neotropical biogeography. Mol Phylogenet Evol 66:
54--68.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2012.09.005}{10.1016/j.ympev.2012.09.005}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{18. Brower AVZ, Wahlberg N, Ogawa JR, Boppr M, Vane-Wright RI (2010)
Phylogenetic relationships among genera of danaine butterflies
(Lepidoptera: Nymphalidae) as implied by morphology and DNA sequences.
Syst Biodivers }\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{299        }& \DIFaddFL{yule  }& \DIFaddFL{0.311199   }& \DIFaddFL{-6.3058    }&          & \emph{\DIFaddFL{Ypthima}}                                             \\
\DIFaddFL{4               }& \DIFaddFL{224        }& \DIFaddFL{yule  }& \DIFaddFL{0.290989   }& \DIFaddFL{-6.2601    }&          & \emph{\DIFaddFL{Charaxes}}                                            \\
\DIFaddFL{5               }& \DIFaddFL{750        }& \DIFaddFL{yule  }& \DIFaddFL{0.186913   }& \DIFaddFL{-147.4146  }&          & \DIFaddFL{Oleriina + Ithomiina + Napeogenina + Dircennina + Godyrina }\\
\DIFaddFL{6               }& \DIFaddFL{405        }& \DIFaddFL{yule  }& \DIFaddFL{0.116252   }& \DIFaddFL{-555.0276  }&          & \DIFaddFL{Satyrinae                                                   }\\
\DIFaddFL{7               }& \DIFaddFL{495        }& \DIFaddFL{yule  }& \DIFaddFL{0.064656   }& \DIFaddFL{-124.9143  }&          & \DIFaddFL{Coenonymphina                                              }\\
\DIFaddendFL 8               \DIFdelbeginFL \DIFdelFL{: 75--89.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1080/14772001003626814}{10.1080/14772001003626814}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{19. Kodandaramaiah U, Lees DC, Mller CJ, Torres E, Karanth KP, et al.
(2010) Phylogenetics and biogeography of a spectacular Old World
radiation of butterflies: the subtribe Mycalesina (Lepidoptera:
Nymphalidae: Satyrini). BMC Evol Biol }\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{609        }& \DIFaddFL{yule  }& \DIFaddFL{0.240562   }& \DIFaddFL{-35.0908   }&          & \DIFaddFL{Phyciodina in part                                         }\\
\DIFaddFL{9               }& \DIFaddFL{787        }& \DIFaddFL{bd    }& \DIFaddFL{0.042332   }& \DIFaddFL{-43.7819   }&          & \DIFaddFL{Danaini in part                                            }\\
\DIFaddendFL 10              \DIFdelbeginFL \DIFdelFL{: 172.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1186/1471-2148-10-172}{10.1186/1471-2148-10-172}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{20. Kodandaramaiah U, Pea C, Braby MF, Grund R, Mller CJ, et al.
(2010) Phylogenetics of Coenonymphina (Nymphalidae: Satyrinae) and the
problem of rooting rapid radiations. Mol Phylogenet Evol 54: 386--394.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2009.08.012}{10.1016/j.ympev.2009.08.012}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{21. Ortiz-Acevedo E, Willmott KR (2013) Molecular systematics of the
butterfly tribe Preponini (Nymphalidae: Charaxinae). Syst Entomol 38:
440--449.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/syen.12008}{10.1111/syen.12008}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{22. De-Silva DL, Day JJ, Elias M, Willmott KR, Whinnett A, et al. (2010)
Molecular phylogenetics of the Neotropical butterfly subtribe Oleriina
(Nymphalidae: Danainae: Ithomiini). Mol Phylogenet Evol 55: 1032--1041.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2010.01.010}{10.1016/j.ympev.2010.01.010}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{23. Freitas AVL, Brown Jr. KS (2004) Phylogeny of the Nymphalidae
(Lepidoptera). Syst Biol 53: 363--383.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1080/10635150490445670}{10.1080/10635150490445670}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{24. Pea C, Wahlberg N, Weingartner E, Kodandaramaiah U, Nylin S, et al.
(2006) Higher level phylogeny of Satyrinae butterflies (Lepidoptera:
Nymphalidae) based on DNA sequence data. Mol Phylogenet Evol 40: 29--49.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2006.02.007}{10.1016/j.ympev.2006.02.007}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{25. Penz CM (1999) Higher level phylogeny for the passion-vine
butterflies (Nymphalidae, Heliconiinae) based on early stage and adult
morphology. Zool J Linn Soc 127: 277--344.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1096-3642.1999.tb00680.x}{10.1111/j.1096-3642.1999.tb00680.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{26. Silva-Brando KL, Wahlberg N, Francini RB, Azeredo-Espin AML, Brown
Jr. KS, et al. (2008) Phylogenetic relationships of butterflies of the
tribe Acraeini (Lepidoptera, Nymphalidae, Heliconiinae) and the
evolution of host plant use. Mol Phylogenet Evol 46: 515--531.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2007.11.024}{10.1016/j.ympev.2007.11.024}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{27. Pea C, Nylin S, Wahlberg N (2011) The radiation of Satyrini
butterflies (Nymphalidae: Satyrinae): a challenge for phylogenetic
methods. Zool J Linn Soc 161: 64--87.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1096-3642.2009.00627.x}{10.1111/j.1096-3642.2009.00627.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{28. Pea C, Nylin S, Freitas AVL, Wahlberg N (2010) Biogeographic
history of the butterfly subtribe Euptychiina (Lepidoptera, Nymphalidae,
Satyrinae). Zool Scr 39: 243--258.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1463-6409.2010.00421.x}{10.1111/j.1463-6409.2010.00421.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{29. Ackery PR (1988) Hostplants and classification: A review of
nymphalid butterflies. Biol J Linn Soc 33: 95--203.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1095-8312.1988.tb00446.x}{10.1111/j.1095-8312.1988.tb00446.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{30. Dyer LA, Gentry GL (2002) Caterpillars and parasitoids of a tropical
lowland wet forest. Available: }%DIFDELCMD < \url{http://www.caterpillars.org}%%%
\DIFdelFL{.
Accessed 17 April 2013.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{31. Beccaloni G, Viloria , Hall S, Robinson G (2008) Catalogue of the
hostplants of the Neotropical butterflies. London: The Natural History
Museum.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{32. Janzen DH, Hallwachs W (2009) Dynamic database for an inventory of
the macrocaterpillar fauna, and its food plants and parasitoids, of Area
de Conservacion Guanacaste (ACG), northwestern Costa Rica. Available:
}%DIFDELCMD < \url{http://janzen.sas.upenn.edu}%%%
\DIFdelFL{. Accessed 17 April 2013.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{33. Core Team R (2013) R: A Language and Environment for Statistical
Computing. Vienna, Austria: R Foundation for Statistical Computing.
Available: }%DIFDELCMD < \url{http://www.r-project.org}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{34. Popescu A-A, Huber KT, Paradis E (2012) ape 3.0: New tools for
distance-based phylogenetics and evolutionary analysis in R.
Bioinformatics 28: 1536--1537.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1093/bioinformatics/bts184}{10.1093/bioinformatics/bts184}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{35. Harmon LJ, Weir JT, Brock CD, Glor RE, Challenger W (2008) GEIGER:
investigating evolutionary radiations. Bioinformatics 24: 129--131.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1093/bioinformatics/btm538}{10.1093/bioinformatics/btm538}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{36. Maddison WP, Midford PE, Otto SP (2007) Estimating a binary
character's effect on speciation and extinction. Syst Biol 56: 701--710.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1080/10635150701607033}{10.1080/10635150701607033}%%%
\DIFdelFL{.}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{37. Willmott KR, Freitas AVL (2006) Higher-level phylogeny of the
Ithomiinae (Lepidoptera: Nymphalidae): classification, patterns of
larval hostplant colonization and diversification . Cladistics 22:
297--368.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1096-0031.2006.00108.x}{10.1111/j.1096-0031.2006.00108.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{38. Scott JA (1986) The Butterflies of North America. Stanford
University Press.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{39. Davis MP, Midford PE, Maddison W (2013) Exploring power and
parameter estimation of the BiSSE method for analyzing species
diversification . BMC Evol Biol }\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{231        }& \DIFaddFL{yule  }& \DIFaddFL{0.209416   }& \DIFaddFL{-4.7955    }&          & \emph{\DIFaddFL{Coenonympha}}                                         \\
\DIFaddFL{11              }& \DIFaddFL{478        }& \DIFaddFL{yule  }& \DIFaddFL{0.311684   }& \DIFaddFL{-9.2218    }&          & \emph{\DIFaddFL{Caeruleuptychia}} \DIFaddFL{+ }\emph{\DIFaddFL{Magneuptychia}}              \\
\DIFaddFL{12              }& \DIFaddFL{659        }& \DIFaddFL{yule  }& \DIFaddFL{0.219253   }& \DIFaddFL{-11.2686   }&          & \emph{\DIFaddFL{Callicore}} \DIFaddFL{+ }\emph{\DIFaddFL{Diaethria}}                        \\
\DIFaddendFL 13              \DIFdelbeginFL \DIFdelFL{: 38.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1186/1471-2148-13-38}{10.1186/1471-2148-13-38}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{40. Litman JR, Danforth BN, Eardley CD, Praz CJ (2011) Why do leafcutter
bees cut leaves? New insights into the early evolution of bees. P Roy
Soc B 278: 3593--3600.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rspb.2011.0365}{10.1098/rspb.2011.0365}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{41. Ryberg M, Matheny PB (2012) Asynchronous origins of ectomycorrhizal
clades of Agaricales. P Roy Soc B 279: 2003--2011.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1098/rspb.2011.2428}{10.1098/rspb.2011.2428}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{42. Brown Jr. KS (1987) Chemistry at the Solanaceae/Ithomiinae
Interface. Ann Mo Bot Gard 74: 359--397.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{43. Brower AVZ, Freitas AVL, Lee M-M}\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{444        }& \DIFaddFL{yule  }& \DIFaddFL{0.220615   }& \DIFaddFL{-43.7812   }&          & \DIFaddFL{Satyrina                                                   }\\
\DIFaddFL{14              }& \DIFaddFL{524        }& \DIFaddFL{yule  }& \DIFaddFL{0.190754   }& \DIFaddFL{-26.0651   }&          & \DIFaddFL{Mycalesina                                                 }\\
\DIFaddFL{15              }& \DIFaddFL{355        }& \DIFaddFL{yule  }& \DIFaddFL{0.234041   }& \DIFaddFL{-6.2013    }&          & \emph{\DIFaddFL{Pedaliodes}}                                          \\
\DIFaddFL{16              }& \DIFaddFL{714        }& \DIFaddFL{yule  }& \DIFaddFL{0          }& \DIFaddFL{0          }&          & \emph{\DIFaddFL{Dryas}} \DIFaddFL{+ }\emph{\DIFaddFL{Dryadula}}                             \\
\DIFaddFL{17              }& \DIFaddFL{377        }& \DIFaddFL{yule  }& \DIFaddFL{0.311671   }& \DIFaddFL{-4.1986    }&          & \emph{\DIFaddFL{Taenaris}}                                            \\
\DIFaddFL{18              }& \DIFaddFL{688        }& \DIFaddFL{yule  }& \DIFaddFL{0.024724   }& \DIFaddFL{-17.5256   }&          & \DIFaddFL{Pseudergolinae                                             }\\
\DIFaddFL{19              }& \DIFaddFL{583        }& \DIFaddFL{yule  }& \DIFaddFL{0          }& \DIFaddFL{0          }& \DIFaddFL{5090.492 }& \emph{\DIFaddFL{Anaeomorpha}} \DIFaddFL{+ }\emph{\DIFaddFL{Hypna}}                                       
\end{tabular}
\begin{flushleft}\DIFaddFL{Split.Node=node number, Model=preferred diversification model by MEDUSA, r=net diversification rate}\DIFaddendFL , \DIFdelbeginFL \DIFdelFL{Silva-Brando KL, Whinnett A, et
al. (2006) Phylogenetic relationships among the Ithomiini (Lepidoptera:
Nymphalidae) inferred from one mitochondrial and two nuclear gene
regions. Syst Entomol 31: 288--301.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1365-3113.2006.00321.x}{10.1111/j.1365-3113.2006.00321.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{44. }%DIFDELCMD < {%%%
\DIFdelFL{FitzJohn}%DIFDELCMD < } %%%
\DIFdelFL{RG, Maddison WP, Otto SP (2009) Estimating
trait-dependent speciation and extinction rates from incompletely
resolved phylogenies. Syst Biol 58: 595--611.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1093/sysbio/syp067}{10.1093/sysbio/syp067}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{45. Wheat CW, Vogel H, Wittstock U, Braby MB, Underwood D, et al. (2007)
The genetic basis of a plant-insect coevolutionary key innovation. P
Natl Acad Sci USA 104: 20427--20431.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1073/pnas.0706229104}{10.1073/pnas.0706229104}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{46. Li W, Schuler MA, Berenbaum MR (2003) Diversification of
furanocoumarin-metabolizing cytochrome P450 monooxygenases in two
papilionids: Specificity and substrate encounter rate. P Natl Acad Sci
USA 100: 14593--14598.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1073/pnas.1934643100}{10.1073/pnas.1934643100}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{47. Bell CD, Soltis DE, Soltis PS (2010) The age and diversification of
the angiosperms re-revisited. Am J Bot 97: 1296--1303.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.3732/ajb.0900346}{10.3732/ajb.0900346}%%%
\DIFdelFL{.
}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{LnLik.part=log likelihood value.
}\end{flushleft}
\end{table}
\DIFaddend 


\DIFdelbegin \DIFdel{48. Somoza R (1998) Updated Nazca (Farallon) - South America relative
motions during the last 40 My: implications for mountain building in the
central Andean region. J S Am Earth Sci }\DIFdelend %DIF >  Table 2
\DIFaddbegin \begin{table}[!h]
    \caption{\bf{Differences in rates estimated by MEDUSA on the MCC
tree from the sample of trees from the posterior distribution and the
MultiMEDUSA approach. Shift consistently recovered across the
sample of trees in bold face.}}
\begin{tabular}{lcrr}
\DIFaddFL{Split.Node }& \DIFaddFL{rate by MEDUSA }& \DIFaddFL{Median rate by MultiMEDUSA }& \DIFaddFL{probability of being recovered }\\
\DIFaddFL{1          }& \DIFaddFL{0.092          }& \DIFaddFL{not found                  }& \DIFaddFL{0.000                          }\\
\DIFaddFL{2          }& \DIFaddFL{0.055          }& \DIFaddFL{-0.030                     }& \DIFaddFL{0.864                          }\\
\bf{3}     & \DIFaddFL{0.184          }& \bf{0.219}                 & \bf{0.961}                     \\
\bf{4}     & \DIFaddFL{0.166          }& \bf{0.212}                 & \bf{0.996}                     \\
\bf{5}     & \DIFaddFL{0.111          }& \bf{0.101}                 & \bf{0.927}                     \\
\DIFaddFL{6          }& \DIFaddFL{0.119          }& \DIFaddFL{0.039                      }& \DIFaddFL{0.131                          }\\
\DIFaddFL{7          }& \DIFaddFL{0.066          }& \DIFaddFL{-0.052                     }& \DIFaddFL{0.195                          }\\
\DIFaddFL{8          }& \DIFaddFL{0.232          }& \DIFaddFL{0.166                      }& \DIFaddFL{0.319                          }\\
\DIFaddFL{9          }& \DIFaddFL{0.042          }& \DIFaddFL{-0.049                     }& \DIFaddFL{0.897                          }\\
\DIFaddFL{10         }& \DIFaddFL{0.058          }& \DIFaddFL{0.149                      }& \DIFaddFL{0.619                          }\\
\DIFaddendFL 11         \DIFdelbeginFL \DIFdelFL{: 211--215.doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/S0895-9811(98)00012-1}{10.1016/S0895-9811(98)00012-1}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{49. Hoorn C, Wesselingh FP, Steege H ter, Bermudez MA, Mora A, et al.
(2010) Amazonia through time: Andean uplift, climate change, landscape
evolution, and biodiversity. Science 330: 927--931.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1126/science.1194585}{10.1126/science.1194585}%%%
\DIFdelFL{.}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{50. Olmstead RG (2013) Phylogeny and biogeography in Solanaceae,
Verbenaceae and Bignoniaceae: a comparison of continental and
intercontinental diversification patterns. Bot J Linn Soc 171: 80--102.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1095-8339.2012.01306.x}{10.1111/j.1095-8339.2012.01306.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{51. Wahlberg N, Rubinoff D (2011) Vagility across Vanessa (Lepidoptera:
Nymphalidae): mobility in butterfly species does not inhibit the
formation and persistence of isolated sister taxa. Syst Entomol 36:
362--370.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1365-3113.2010.00566.x}{10.1111/j.1365-3113.2010.00566.x}%%%
\DIFdelFL{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelFL{52. Mller CJ, Wahlberg N, Beheregaray LB (2010) `After Africa': the
evolutionary history and systematics of the genus Charaxes Ochsenheimer
(Lepidoptera: Nymphalidae) in the Indo-Pacific region. Biol J Linn Soc
100: 457--481.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1111/j.1095-8312.2010.01426.x}{10.1111/j.1095-8312.2010.01426.x}%%%
\DIFdelFL{.}\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{0.311          }& \DIFaddFL{0.208                      }& \DIFaddFL{0.831                          }\\
\bf{12}    & \DIFaddFL{0.219          }& \bf{0.135}                 & \bf{0.911}                     \\
\DIFaddFL{13         }& \DIFaddFL{0.082          }& \DIFaddFL{0.117                      }& \DIFaddFL{0.276                          }\\
\DIFaddFL{14         }& \DIFaddFL{0.099          }& \DIFaddFL{0.115                      }& \DIFaddFL{0.379                          }\\
\DIFaddFL{15         }& \DIFaddFL{0.113          }& \DIFaddFL{0.127                      }& \DIFaddFL{0.825                          }\\
\DIFaddFL{16         }& \DIFaddFL{0.222          }& \DIFaddFL{-0.005                     }& \DIFaddFL{0.659                          }\\
\DIFaddFL{17         }& \DIFaddFL{0.243          }& \DIFaddFL{0.248                      }& \DIFaddFL{0.785                          }\\
\DIFaddFL{18         }& \DIFaddFL{0.192          }& \DIFaddFL{-0.064                     }& \DIFaddFL{0.024                          }\\
\DIFaddFL{19         }& \DIFaddFL{0.064          }& \DIFaddFL{-0.087                     }& \DIFaddFL{0.381                         
}\end{tabular}
\end{table}
\DIFaddend 


\DIFdelbegin \DIFdel{53. Aduse-Poku K, Vingerhoedt E, Wahlberg N (2009) Out-of-Africa again:
a phylogenetic hypothesis of the genus Charaxes (Lepidoptera:
Nymphalidae) based on }\DIFdelend %DIF >  Table 3
\DIFaddbegin \begin{table}[!h]
    \caption{\bf{Likelihood ratio test between the model of increased
diversification of nymphalids feeding on Solanaceae against a model
forcing equal speciation rates (no effect on diversification).}}
\begin{tabular}{lcrrlr}
\DIFaddFL{Df           }& \DIFaddFL{lnLik }& \DIFaddFL{AIC     }& \DIFaddFL{ChiSq  }& \multicolumn{1}{r}{p} &         \\
\DIFaddFL{full         }& \DIFaddFL{6     }& \DIFaddFL{-1613.3 }& \DIFaddFL{3238.5 }&                       &         \\
\DIFaddFL{equal.lambda }& \DIFaddendFL 5     \DIFdelbeginFL \DIFdelFL{gene regions. Mol Phylogenet Evol 53: 463--478.
doi:}%DIFDELCMD < \href{http://dx.doi.org/10.1016/j.ympev.2009.06.021}{10.1016/j.ympev.2009.06.021}%%%
\DIFdelFL{.
}\DIFdelendFL \DIFaddbeginFL & \DIFaddFL{-1619.4 }& \DIFaddFL{3248.9 }& \DIFaddFL{12.3                  }& \DIFaddFL{0.00045
}\end{tabular}
\end{table}
\DIFaddend 




\end{document}
