# Response to reviewers

# Reviewer 1

**1. Reviewer's Comment:**

Introduction
1. Second paragraph, second sentence: The fact that the Nymphalids are the “largest family” of butterflies conveys an exaggerated importance of their diversity based on idiosyncrasy of Linnean taxonomy (the sister group to Nymphalidae, the clade that includes riodinids and lycaenids, has over 6500 species). Consider revising the sentence, omitting superlatives (“biggest”, “largest”, “most”, etc.).

**1. Response:**

> We have rewritten the sentence to:
> "Nymphalidae contains around 6000 species [6], and several members are
considered model organisms in evolutionary biology [7–9]."

**2. Reviewer's Comment:**

The authors should mention that the ideal approach would accommodate uncertainty by sampling from a distribution of trees which reflected the variation in confidence in various parts of the tree (such as a Bayesian posterior distribution or a bootstrap sample). See Holder,M. and Lewis,P.O. (2003) Phylogeny estimation: traditional and Bayesian approaches. Nat. Rev. Genet., 4, 275–284

**2. Response:**

> We have specified that the sample of trees was taken from the Bayesian run in
Wahlberg et al., 2009.

**3. Reviewer's Comment:**

Middle of paragraph (and throughout the manuscript): revise the tree sampling to “...therefore randomly sampled 1000 trees from the posterior distribution”; in the remainder of the manuscript, refer to these as the “1000 sampled trees”, not “1000 random trees”, which can take on a very different meaning.

**3. Response:**

> We have corrected some sentences in the manuscript to specify that we are
referring to the same set of 1000 trees.

**4. Reviewer's Comment:**

Define what 'MuSSE' refers to (Multiple State Speciation and Extinction?).

**4. Response:**

> We have included the definition of MuSSE in the manuscript.


**5. Reviewer's Comment:**

A reference for the statement “most of Nymphalidae butterflies are restricted to use one plant family as hostplant” is warranted.

**5. Response:**

> We have included the numbers of single-family feeders based on calculations
of our own data on hostplant use across Nymphalidae genera. The text has been
modified accordingly:

> "Because most of Nymphalidae butterflies are restricted to use one plant
family as hostplant (232 genera use only one plant family versus 176 that use
        more than one; based on our data in supp. mat. 15), the ..."

**6. Reviewer's Comment:**

At the end of this paragraph, the authors indicated they tested the effect of character coding on their analyses. A more explicit description of the test would be useful, along with predictions of how results would be effected (i.e. how would a large effect vs. no effect of coding Hypanartia and Vanessa as Solanaceae feeders manifest in the results?).

**6. Response:**

> We included the following sentence to clarify the issue:  
**"We expected that
coding these two genera as either absence or presence would not have a
significant effect on our results."**

**7, 8, 9, 10. Reviewer's Comment:**

7. First paragraph, second sentence: change the tense of “run” to “ran”.
8. Second paragraph, third sentence: replace “Thus” with “Furthermore”.
9. There appears to be an orphaned right-parenthesis at the end of the second paragraph.
10. Third paragraph, last sentence: The node number is not really relevant, so beginning of the sentence could be revised to “For example, the Charaxes + Polyura clade was found in only 256 trees...”

**7, 8, 9, 10. Response:**

> We made the suggested corrections.

**11. Reviewer's Comment:**

Fourth paragraph: The point of the paragraph is not clear. What rates are being
compared? MEDUSA vs. MultiMEDUSA? Consider revising to make the point clearer
or omit the sentence.

**11. Response:**

> We have rewritten the sentence and now is more clear:  
**"For the diversification shifts found in both the MCC tree and most of the
samples of 1000 trees (frequency more than 90%; Table 2), the MultiMEDUSA
approach recovered different rates of diversification than
those found using when the MCC tree alone."**.

**12. Reviewer's Comment:**

Fifth paragraph, first sentence (and potentially elsewhere): Consider using a consistent means of presenting probability values. In this sentence, the authors refer to “90% of probability in the trees”, but in the preceding paragraph, the authors present probabilities as a decimal (“probability of being recovered higher than 0.90”). Granted, these two probabilities refer to different events, but perhaps it would be best to refer to them in a consistent manner, such as “0.90 posterior probability”.

**12. Response:**

> We have corrected this and normalized the probability values to decimals in the manuscript.

**13. Reviewer's Comment:**
The third sentence (length of analysis and burnin details) should be moved to the methods section.

**13. Response:**

> We made the suggested corrections.

**14. Reviewer's Comment:**

Mid-way through paragraph, discussion of likelihood ratio test: The source of this P-value needs to be described (chi-squared approximation with 1 degree of freedom?), here or in the methods section (see p. 617 in Pagel 1999 Syst. Biol. 48:612-622).

**14. Response:**

> We have include the statistics suggested by the reviewer: 
"**$\chi^2 = 12.3; 1 df; p < 0.001$**".

**15. Reviewer's Comment:**

The closing three sentences (about BiSSE performance) should be moved to the methods section.

**15. Response:**

> We made the suggested corrections.

**16. Reviewer's Comment:**

First paragraph: The authors' assessment of analyzing diversification shifts using a single tree, without accommodating our level of certainty in relationships, is justified; however, MEDUSA itself, that is, the approach using stepwise AIC to identify shifts in diversification rate, should not be the target of criticism. After all, MultiMEDUSA is doing the same thing, but on a sample of trees. Revise this paragraph with a more appropriate target for the criticism (i.e. analyses performed on single trees).

**16. Response:**

> We have reworded the paragraph to avoid blaming the method:

> "The MEDUSA method has been used to infer changes in net diversification rates in a phylogenetic
tree. Since its publication [16] the results of using MEDUSA on a single tree, the maximum clade
credibility tree, have been used for generation of hypotheses and discussion [10,42,43]. However, different
diversification shifts and different rates of diversification are found for certain lineages when phylogenetic
uncertainty was taken into account by using MEDUSA on a random sample of trees from the posterior
distribution of a Bayesian run. We found that some diversification splits, estimated on the Nymphalidae
maximum clade credibility tree, were found with very low probability in the random sample of 1000 trees
from the posterior distribution (Fig. 3, Table 2). We also found that, even though the analyses estimated
the same diversification splits on two or more trees, the estimated net diversification rates could vary
widely (Fig. 2)."

**17. Reviewer's Comment:**

Seventh paragraph, second to last sentence: Two ages for Ithomiini are given – are they the origin and diversification ages? If so, revise the sentence to include this information (rather than requiring readers to refer to the preceding sentence): “Reference 10 gives the ages of origin and diversification for Ithomiini at 45 (39-53) and 37 (32-43) MYA, respectively”.

**17. Response:**

> We have followed the reviewer's suggestion.

**18. Reviewer's Comment:**

Eighth paragraph: This paragraph discussing biogeography does not fit and could be omitted without detriment to the manuscript.

**18. Response:**

> We have followed the reviewer's suggestion and removed this paragraph.

**19. Reviewer's Comment:**

First paragraph, first sentence: replace “gave” with “showed”.

**19. Response:**

> We corrected the text following the reviewer's suggestion.

**20. Reviewer's Comment:**

This section is predominantly speculative. Consider a more rigorous discussion (if appropriate) or omit.

**20. Response:**

> We have followed the reviewer's suggestion and removed this paragraph.

**21. Reviewer's Comment:**

First paragraph, second sentence: revise “13% of probabilities of the trees from the posterior distribution” to “13% of the trees from the posterior distribution”.

**21. Response:**

> We corrected the text following the reviewer's suggestion.

**22. Reviewer's Comment:**

Second paragraph: this paragraph is, for the most part, unnecessary. The last sentence of the paragraph should be revised and appended to the end of the preceding (first) paragraph: “...not picked up by MEDUSA. This would be expected if the diversification of Satyrinae occurred in a stepwise manner, with pulses or bursts of diversification for certain lineages but unlikely for the tribe Satyrini as a whole”.

**22. Response:**

> We corrected the text following the reviewer's suggestion.

**23. Reviewer's Comment:**

Final two paragraphs: There needs to be a section break, to indicate a change in the discussion topic. “Conclusions” would be appropriate to indicate to readers that these paragraphs represent a summary of the major points of the work. The final paragraph is a good summary of the work, and it would be a disservice to have readers miss it because it appears under the 'Satyrini' section.

**23. Response:**

> As suggested by the reviewer, he have created a **Conclusions** section for
the last two paragraphs of the text.

**24. Reviewer's Comment:**

Figure 1: The legend needs to indicate what the node labels (the circled integers) and the decimal values in the tree indicate.

**24. Response:**

> We have appended the following text to the legend of figure 1:  
**"Circles on nodes indicate the diversification shift number as found by MEDUSA.
Numbers next to circles indicat the posterior probability values for
such nodes."**

**25. Reviewer's Comment:**

Figure 2: Omit the legend in the figure (“Boxplot of diversification...”) and revise the legend to read: “Diversification rates for taxa estimated by MEDUSA on the sample of 1000 trees.”

**25. Response:**

> We removed the legend from the figure file and rewrote the legend in the MS
as suggested by the reviewer.

**26. Reviewer's Comment:**

Figure 3: Omit the legend in the figure (“Probability of finding nodes...”) and revise the legend to read: “Results of MultiMEDUSA analysis showing the probability of specific nodes being characterized by significant shifts in diversification rate.”

**26. Response:**

> We removed the legend from the figure file and rewrote the legend in the MS
as suggested by the reviewer.

**27. Reviewer's Comment:**

Figure 4: Describe what lambda ($\lambda$) and $r$ represent in the figure.

**27. Response:**

> We included the following clarification in the legend: **"(speciation rate =
$\lambda1$, net diversification rate = $r1$)."**

**28, 29. Reviewer's Comment:**

Supporting_Information_S1.nex: remove bolded “Genus.”

**28, 29. Response:**

> We followed the reviewer's suggestions and included better descriptions for
the supplementary materials.


# Reviewer 2

**1. Reviewer's Comment:**

The authors need to provide more information on what cutoff they used and why.

**1. Response:**

> Our manuscript already includes information about the AIC value used for
choosing the best model. The following text appears in our original submission
in the Methods-> Detecting diversification shifts on phylogenetic trees->
last paragraph:  
>**"We used the AICc threshold of 7.8 units, as estimated by MEDUSA,
as the limit for a significantly better fit to select among increasingly
complex alternative models."**

**2. Reviewer's Comment:**

I’m not sure discussing the limitations of MEDUSA, or the tendency for MEDUSA
to give inconsistent results across post burn-in trees, is really valid based
on one dataset. That is, the unit of replication is the alignment – therefore
the authors have effectively one data point and zero degrees of freedom to say
much of anything about the power of MEDUSA across the post burnin sample. I
think if the authors want to explore the power of MEDUSA, they need many
different alignments (real or simulated). As it is now, I don’t think they
really have the data to say anything about MEDUSA in general, and only about
MEDUSA in the context of the Nymphalid tree. They need to at least point out
there inability to generalize in the manuscript.
 
**2. Response:**

> * As reviewer 1 suggested, our text gave the impression that we were
criticizing the method MEDUSA. We have rewritten the following sections to
clarify our criticism **that relying on the inferences made on only one tree
(the MCC tree) is risky as analyses of equally good hyphoteses (other trees from the
posterior distribution) cand potentially return different results**:

> * Abstract  
"In evolutionary biology, analysis of maximum credibility trees in the software
MEDUSA (modelling evolutionary diversity using stepwise AIC) is a popular
method for estimation of shifts in diversification rates.
We investigated whether phylogenetic uncertainty can produce different results
by extending the method across a 
random sample of trees from the posterior distribution of a Bayesian run."

> * Results -> Phylogenetic uncertainty in the MultiMEDUSA approach
We changed the sentence "We tested MEDUSA..." to "We used MEDUSA to find out...".

> * Also, as suggested by the reviewer 1 (point 16 lines above) we improved the
discussion.

> * We have added the following in the Conclusions to acknowledge that the
reported behaviour might also occur in analyses of other taxa:  
> " However, by using a
MultiMEDUSA approach, we found that for this Nymphalidae dataset some of these
splits might be greatly
affected by phylogenetic uncertainty.
> We recommend that all datasets should be analyzed using both approaches, MEDUSA
and MultiMEDUSA, in order to test whether the results are robust when
phylogenetic uncertainty is taken into account."

**3. Reviewer's Comment:**

“Our results show that taking phylogenetic uncertainty into account when estimating diversification rate shifts is of great importance, and relying on the maximum credibility tree alone potentially can give erroneous results”. The results are not really in error. The results are the correct results for the model. The real question is whether it is the appropriate model.

**3. Response:**

> We have rewritten that sentence in the abstract as follows:  
"Our results show that taking phylogenetic uncertainty into account when
estimating net diversification rate shifts is of great importance, as very
different results can be obtained when using the maximum clade credibility tree
and other trees from the posterior distribution."

**4. Reviewer's Comment:**

Why are they restricting their analysis to only one method (a method that they point out might not be reliable)? There are alternative methods available, and it might be interesting to see if consistent results are observed (eg., RC, PRC, BAMM). There is very little development on other aspects of plant – herbivore interactions and how they might be predicted to affect diversification rate.

**4. Response:**

[**>>>> Marianne**, can you write whey we are not using the RC and PRC methods? "The
parametric rate comparison and relative rates test that he suggests both
require incomplete taxon sampling to be nonrandom, which ours definitely isn't
so we can easily explain why we didn't use these."**<<<<<<**]

The method that we used, BiSSE, might not be reliable when the number of taxa
is small, and there are biases in the number of species that have each of the
character states. However, we stated in the manuscript that according to the
evaluation of BiSSE by @davis2013, our data should not be affected for such
biases.

The method BAMM can be used to explore the evolution of phenotypic traits. 
However the trait characters
[**must be continuous**](http://bamm-project.org/configuration.html#id5).
In our case the "feeding on Solanaceae plants" trait is categorical 
(absence/presence) so it cannot be analyzed with BAMM.

BAMM also measures the diversification rates of
particular clades. We run an analysis to test whether we obtained congruent
results. In the Fig. 1, shown below, we can see that BAMM was able to pick up two
increases in diversification for Ithomiini butterflies, one for the origin of
the clade and, most importantly, another increase during the recent
diversification of the group. This is in agreement with our results from using
BiSSE, that the ithomiines using Solanaceae underwent an increase in
diversification rates.

![Bamm analysis](response_to_reviewers_fig1.pdf)

**5. Reviewer's Comment:**

The authors should ‘play’ with an AIC cutoff, simply to explore how it might affect there inferences.

**5. Response:**

> As specified lines above and in the manuscript, we used the AIC cutoff value
7.8 as suggested by MEDUSA (this value is regarded as optimal due to the number
of taxa in our analyses).
Changing the cutoff value would allow fewer or additional diversification
shifts to be picked up by MEDUSA. Hovewer the initial diversification shifts
will remain the same unless the AIC cutoff used is so hight that no single
diversification shifts are recovered.
Thus playing with the AIC cutoff values does not affect most of the
diversification shifts discussed in our manuscript. 
We rerun all our MEDUSA and MultiMEDUSA analysis with differente AIC cutoff
values to see any effect on the results. We used the AIC values 4 and 11.

> The results in Fig.2 (also attached as PDF file) shows our MEDUSA results from
the manuscript (cutoff 7.8) along with results with AIC cutoff values 4 and 11.
We see that the first 19 diversification shitfs are the same for cutoff values
7.8 and 4. However when using AIC cutoff 4 we recovered additional
diversification shifts as the software kept trying to fit additional models
until the differente in AIC values was less than 4. As expected MultiMEDUSA
also found additinal diversification shifts across the random sample of 1000
trees.

> When we used AIC cutoff 11, we obtained fewer diversification shifts. We
recovered only 16, which are the same first 16 that we found in our analysis
with AIC 7.8.

![MEDUSA analyses under three AIC values](multimedusa_7_8_vs_4_vs_11_AIC_cutoff.pdf)

**6. Reviewer's Comment:**

In addition, the authors should be cautious about interpreting AIC in a
frequentist manner (e.g., “Only three out of 13 significant shifts found on the
maximum credibility tree were consistent across more than 95% of the
trees from the posterior”) – that is, AIC is a model selection tool,
which should act as a guide for selecting the ‘best’ family of
models. Delta AIC should really not be thought of as alpha = 0.05.

**6. Response:**

> We agree that AIC values should not be treated in a frequentist manner.
However, we use the term **"significant shifts"**, in the same way as the authors of the
method MEDUSA [@alfaro2009], to pinpoint to specific nodes that show a higher
net diversification rate than ancestral or descendant nodes.

> We also used the AIC value in the correct way. We used the cutoff 7.8 for all
MEDUSA analyses. Then we counted the number of significant shifts infered in
each tree and tried to pinpoint those that were consistently recovered across
the sample of 1000 random trees (and are also present in the MCC tree).
Since no single significant shift was found in all 1000 trees we wanted to show
those are were most commonly recovered. This is way we considered consistent
diversification shifts those that were found in most of the trees (in more than
95% of them).

**7. Reviewer's Comment:**

Why are authors referred to as references, rather than by authors.

As noted by Reference 36, the responsible trait might...

**7. Response:**

> We have fixed those references as suggested by the reviewer.

# References
